{
  "nbformat": 4,
  "nbformat_minor": 5,
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.7.9"
    },
    "colab": {
      "provenance": []
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# ‚úçÔ∏èüìú Cadmus\n",
        "\n"
      ],
      "metadata": {
        "id": "GDdXNIoPnw89"
      },
      "id": "GDdXNIoPnw89"
    },
    {
      "cell_type": "markdown",
      "source": [
        "This project aims to build an automated full-text retrieval system for the generation of large biomedical corpora from published literature for research purposes. Cadmus has been developed for use in non-commercial research. Use out with this remit is not recommended, nor is the intended purpose.\n",
        "\n",
        "GitHub: https://github.com/biomedicalinformaticsgroup/cadmus"
      ],
      "metadata": {
        "id": "-_f20ckhn7Bl"
      },
      "id": "-_f20ckhn7Bl"
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "regular-crystal"
      },
      "source": [
        "# üìã Requirements\n"
      ],
      "id": "regular-crystal"
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "narrow-thing"
      },
      "source": [
        "In order to run the code, you need a few things:\n",
        "\n",
        "You need to have Java 7+.\n",
        "\n",
        "You need to git clone the project and install it.\n",
        "\n",
        "An API key from NCBI (this is used to search PubMed for articles using a search string or list of PubMed IDs; you can find more information [here](https://ncbiinsights.ncbi.nlm.nih.gov/2017/11/02/new-api-keys-for-the-e-utilities/)).\n",
        "\n",
        "*In case you are running cadmus on a shared machine, you need to terminate all the Tika instances present in the tmp directory if you are not the owner of the instances, so cadmus can restart them for you.*\n",
        "\n",
        "**Recommended requirements:**\n",
        "\n",
        "An API key from Wiley, this key will allow you to get access to the OA and publications you or your institution has the right to access from Wiley. You can find more information [here](https://onlinelibrary.wiley.com/library-info/resources/text-and-datamining)\n",
        "\n",
        "An API key from Elsevier, this key will allow you to get access to the OA and publications you or your institution has the right to access from Elsevier. You can find more information [here](https://dev.elsevier.com/)\n"
      ],
      "id": "narrow-thing"
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "TqFkKiJXcSOI"
      },
      "source": [
        "# ‚úçÔ∏èüìú Import Cadmus"
      ],
      "id": "TqFkKiJXcSOI"
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "cordless-display"
      },
      "source": [
        "This notebook is using the following version of Python"
      ],
      "id": "cordless-display"
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "moved-anchor",
        "outputId": "77667fce-f31f-4e14-de89-f901e13e10c1"
      },
      "source": [
        "!python --version"
      ],
      "id": "moved-anchor",
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Python 3.12.12\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "weighted-sweden",
        "collapsed": true
      },
      "source": [
        "!git clone https://github.com/biomedicalinformaticsgroup/cadmus.git"
      ],
      "id": "weighted-sweden",
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "cqygbB_Pc_ML",
        "collapsed": true
      },
      "source": [
        "!pip install ./cadmus"
      ],
      "id": "cqygbB_Pc_ML",
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "complicated-taxation"
      },
      "source": [
        "# üìö Import libraries"
      ],
      "id": "complicated-taxation"
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "joined-mediterranean"
      },
      "source": [
        "from cadmus import bioscraping\n",
        "from cadmus import parsed_to_df\n",
        "import zipfile\n",
        "import json\n",
        "import pandas as pd"
      ],
      "id": "joined-mediterranean",
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "opposed-opening"
      },
      "source": [
        "# üöÄ Retrieving a corpus using search terms (without subscriptions)"
      ],
      "id": "opposed-opening"
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "re7Mdl5d8rwq"
      },
      "source": [
        "The format we are using for the search term(s) is the same as the one for [PubMed](https://pubmed.ncbi.nlm.nih.gov/). You can first try your search term(s) on PubMed and then use the same search term(s) as input for cadmus `bioscraping`.\n",
        "\n",
        "In order to create your corpora, you are going to use the function called `bioscraping`. The function is taking the following required parameters:\n",
        "\n",
        "1. A PubMed query string or a Python list of PubMed IDs\n",
        "2. An email address\n",
        "3. Your NCBI_API_KEY\n",
        "   \n",
        "The function can also receive optional parameters.\n",
        "\n",
        "1. wiley_api_key parameter allows Wiley to identify which publications you or your institution have the right to access. It will give you access to the OA publications that you would not get access to without the key. **RECOMMENDED**\n",
        "2. elsevier_api_key parameter allows Elsevier to identify which publications you or your institution have the right to access. It will give you access to the OA publications that you would not normally have access to without the key. **RECOMMENDED**\n",
        "3. The \"start\" parameter tells the function at which service we were before failure (e.g. crossref, doi, PubMed Central API, ...).\n",
        "4. The \"idx\" parameter tells the function what is the last saved row index (article).\n",
        "\n",
        "Start and idx are designed to be used when restarting cadmus after a program failure. When Cadmus is running, there is a repeated output feed at the top of the live output.  This line will show you the stage and index that your output dataframe was last saved in case of failure for whatever reason. By using these optional parameters, the program will pick off where it left off, saving you from starting the process from the beginning again.\n",
        "\n",
        "5. \"full_search\", in case you want to check if a document became available since the last time you tried. \"full_search\" has three predefined values:\n",
        "\n",
        "    - The default value is 'None'; the function only looks for the new articles since the last run.\n",
        "    - 'light', the function looks for the new articles since the last run and retried the row where we did not get any format.\n",
        "    - 'heavy', the function looks for the new articles since the last run and retried the row where it did not retrieve at least one tagged version (i.e. HTML or XML) in combination with the PDF format.  \n",
        "\n",
        "6. The \"keep_abstract\" parameter has the default value 'True' and can be changed to 'False'. When set to 'True', our parsing will load any format from the beginning of the document. If changes to 'False', our parsing is trying to identify the abstract from any format and starts to extract the text after it. We are offering the option of removing the abstract, but we can not guarantee that our approach is more reliable for doing so. In case you would like to apply your own parsing method for removing the abstract, feel free to load any file saved during the retrieval available in the output folder:\n",
        "```\"output/formats/{format}s/{index}.{suffix}.zip\"```.  "
      ],
      "id": "re7Mdl5d8rwq"
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "structural-saskatchewan",
        "outputId": "68b6a7e1-3462-4cfb-9c25-98bef95f0cdc"
      },
      "source": [
        "pmids = ['19902024', '28089709', '28601864', '31138673', '31605240']\n",
        "\n",
        "bioscraping(pmids,\n",
        "            EMAIL, #You need to insert your email address here\n",
        "            NCBI_API_KEY, #You need to insert your NCBI_API_KEY here\n",
        "            wiley_api_key = WILEY_API_KEY, #This is an optional parameter.\n",
        "            #You can insert your WILEY_API_KEY here\n",
        "            elsevier_api_key = ELSEVIER_API_KEY, #This is an optional parameter.\n",
        "            #You can insert your ELSEVIER_API_KEY here\n",
        "            colab1 = True # This parameter is for this Notebook example only.\n",
        "            #It bypass the edirect module that can not be run on Colab.\n",
        "            )"
      ],
      "id": "structural-saskatchewan",
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Result for retrieved_df : \n",
            "Here is the performance thus far:\n",
            "PDF:3 = 60.0%\n",
            "XML:3 = 60.0%\n",
            "HTML:0 = 0.0%\n",
            "Plain Text:1 = 20.0%\n",
            "\n",
            "We have a tagged version (HTML, XML) for 3 articles = 60.0%\n",
            "\n",
            "We only have the abstract but not the associated content text for 1 articles = 20.0%\n",
            "\n",
            "We have a content text for 4 out of 5 articles = 80.0%\n",
            "\n",
            "\n",
            "Result for retrieved_df2 : \n",
            "Here is the performance thus far:\n",
            "PDF:3 = 60.0%\n",
            "XML:3 = 60.0%\n",
            "HTML:0 = 0.0%\n",
            "Plain Text:1 = 20.0%\n",
            "\n",
            "We have a tagged version (HTML, XML) for 3 articles = 60.0%\n",
            "\n",
            "We only have the abstract but not the associated content text for 1 articles = 20.0%\n",
            "\n",
            "We have a content text for 4 out of 5 articles = 80.0%\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "shaped-minister"
      },
      "source": [
        "with zipfile.ZipFile(\"./output/retrieved_df/retrieved_df2.json.zip\", \"r\") as z:\n",
        "    for filename in z.namelist():\n",
        "        with z.open(filename) as f:\n",
        "            data = f.read()\n",
        "            data = json.loads(data)\n",
        "\n",
        "\n",
        "f.close()\n",
        "z.close()\n",
        "metadata_retrieved_df = pd.read_json(data, orient='index')\n",
        "metadata_retrieved_df.pmid = metadata_retrieved_df.pmid.astype(str)"
      ],
      "id": "shaped-minister",
      "execution_count": 7,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "confident-monte"
      },
      "source": [
        "There are 5 publications records and possibly full-texts extracted from the query."
      ],
      "id": "confident-monte"
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "disturbed-airline",
        "outputId": "ed287067-2adc-48b2-cf26-137661c32f7b"
      },
      "source": [
        "metadata_retrieved_df.shape"
      ],
      "id": "disturbed-airline",
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(5, 25)"
            ]
          },
          "metadata": {},
          "execution_count": 8
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 706
        },
        "id": "printable-packet",
        "outputId": "7be23194-6cb8-43d8-852a-8cb7470b7334"
      },
      "source": [
        "metadata_retrieved_df"
      ],
      "id": "printable-packet",
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                      pmid       pmcid  \\\n",
              "7575147222f842aba0727e3632c8f6dc  19902024  PMC2771255   \n",
              "8677a2c0e6214547acca432c72af98a2  28089709  PMC5357736   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  28601864  PMC5644969   \n",
              "0b80f7eaaba9488dba9af951a818e062  31138673  PMC6538848   \n",
              "8ff3e24cb260420498b5537e372a1a1e  31605240  PMC7165363   \n",
              "\n",
              "                                                                              title  \\\n",
              "7575147222f842aba0727e3632c8f6dc  Changes on the physiological lactonase activit...   \n",
              "8677a2c0e6214547acca432c72af98a2  Objective assessment of dietary patterns by us...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  Effect of a High-Protein Diet versus Standard-...   \n",
              "0b80f7eaaba9488dba9af951a818e062  A Multi-omics Approach to Unraveling the Micro...   \n",
              "8ff3e24cb260420498b5537e372a1a1e  Elevated serum ceramides are linked with obesi...   \n",
              "\n",
              "                                                                           abstract  \\\n",
              "7575147222f842aba0727e3632c8f6dc  Low caloric diet (LCD) is used for weight loss...   \n",
              "8677a2c0e6214547acca432c72af98a2  BACKGROUND: Accurate monitoring of changes in ...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  BACKGROUND: Some studies have shown that prote...   \n",
              "0b80f7eaaba9488dba9af951a818e062  Long-term consumption of dietary fiber is gene...   \n",
              "8ff3e24cb260420498b5537e372a1a1e  INTRODUCTION: Low gut microbiome richness is a...   \n",
              "\n",
              "                                                                               mesh  \\\n",
              "7575147222f842aba0727e3632c8f6dc                                               None   \n",
              "8677a2c0e6214547acca432c72af98a2  [Adult, Biomarkers/*urine, Cross-Over Studies,...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  [Adult, Biomarkers/*blood, Blood Glucose/analy...   \n",
              "0b80f7eaaba9488dba9af951a818e062                                               None   \n",
              "8ff3e24cb260420498b5537e372a1a1e  [Adult, Ceramides/*blood/metabolism, Chromatog...   \n",
              "\n",
              "                                                                           keywords  \\\n",
              "7575147222f842aba0727e3632c8f6dc  [BMI, high-density lipoprotein, low caloric di...   \n",
              "8677a2c0e6214547acca432c72af98a2                                               None   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  [Diet, Metabolic syndrome, Protein intake, Wei...   \n",
              "0b80f7eaaba9488dba9af951a818e062  [AXOS, dietary fiber, glucose homeostasis, lip...   \n",
              "8ff3e24cb260420498b5537e372a1a1e  [Ceramides, Endotoxin, Glucose metabolism, Mic...   \n",
              "\n",
              "                                                                            authors  \\\n",
              "7575147222f842aba0727e3632c8f6dc  [Kotani K, Sakane N, Sano Y, Tsuzaki K, Matsuo...   \n",
              "8677a2c0e6214547acca432c72af98a2  [Garcia-Perez I, Posma JM, Gibson R, Chambers ...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64         [Campos-Nonato I, Hernandez L, Barquera S]   \n",
              "0b80f7eaaba9488dba9af951a818e062  [Benitez-Paez A, Kjolbaek L, Gomez Del Pulgar ...   \n",
              "8ff3e24cb260420498b5537e372a1a1e  [Kayser BD, Prifti E, Lhomme M, Belda E, Dao M...   \n",
              "\n",
              "                                                                            journal  \\\n",
              "7575147222f842aba0727e3632c8f6dc     Journal of clinical biochemistry and nutrition   \n",
              "8677a2c0e6214547acca432c72af98a2               The lancet. Diabetes & endocrinology   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                      Obesity facts   \n",
              "0b80f7eaaba9488dba9af951a818e062                                           mSystems   \n",
              "8ff3e24cb260420498b5537e372a1a1e  Metabolomics : Official journal of the Metabol...   \n",
              "\n",
              "                                                                           pub_type  \\\n",
              "7575147222f842aba0727e3632c8f6dc                                  [Journal Article]   \n",
              "8677a2c0e6214547acca432c72af98a2  [Journal Article, Randomized Controlled Trial,...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64     [Journal Article, Randomized Controlled Trial]   \n",
              "0b80f7eaaba9488dba9af951a818e062                                  [Journal Article]   \n",
              "8ff3e24cb260420498b5537e372a1a1e  [Journal Article, Research Support, Non-U.S. G...   \n",
              "\n",
              "                                    pub_date  ... pdf xml  html plain pmc_tgz  \\\n",
              "7575147222f842aba0727e3632c8f6dc  2009-11-24  ...   1   1     0     0       1   \n",
              "8677a2c0e6214547acca432c72af98a2  2017-03-24  ...   1   0     0     1       1   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  2017-11-24  ...   0   0     0     0       0   \n",
              "0b80f7eaaba9488dba9af951a818e062  2019-05-28  ...   1   1     0     0       1   \n",
              "8ff3e24cb260420498b5537e372a1a1e  2019-10-11  ...   0   1     0     0       0   \n",
              "\n",
              "                                                                        xml_parse_d  \\\n",
              "7575147222f842aba0727e3632c8f6dc  {'file_path': './output/formats/xmls/757514722...   \n",
              "8677a2c0e6214547acca432c72af98a2                                                 {}   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "0b80f7eaaba9488dba9af951a818e062  {'file_path': './output/formats/xmls/0b80f7eaa...   \n",
              "8ff3e24cb260420498b5537e372a1a1e  {'file_path': './output/formats/xmls/8ff3e24cb...   \n",
              "\n",
              "                                  html_parse_d  \\\n",
              "7575147222f842aba0727e3632c8f6dc            {}   \n",
              "8677a2c0e6214547acca432c72af98a2            {}   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64            {}   \n",
              "0b80f7eaaba9488dba9af951a818e062            {}   \n",
              "8ff3e24cb260420498b5537e372a1a1e            {}   \n",
              "\n",
              "                                                                        pdf_parse_d  \\\n",
              "7575147222f842aba0727e3632c8f6dc  {'file_path': './output/formats/pdfs/757514722...   \n",
              "8677a2c0e6214547acca432c72af98a2  {'file_path': './output/formats/pdfs/8677a2c0e...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "0b80f7eaaba9488dba9af951a818e062  {'file_path': './output/formats/pdfs/0b80f7eaa...   \n",
              "8ff3e24cb260420498b5537e372a1a1e                                                 {}   \n",
              "\n",
              "                                                                      plain_parse_d  \\\n",
              "7575147222f842aba0727e3632c8f6dc                                                 {}   \n",
              "8677a2c0e6214547acca432c72af98a2  {'file_path': './output/formats/txts/8677a2c0e...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "0b80f7eaaba9488dba9af951a818e062                                                 {}   \n",
              "8ff3e24cb260420498b5537e372a1a1e                                                 {}   \n",
              "\n",
              "                                  content_text  \n",
              "7575147222f842aba0727e3632c8f6dc             1  \n",
              "8677a2c0e6214547acca432c72af98a2             1  \n",
              "e937a9417a6e4723bdb6d07e4cd71a64             0  \n",
              "0b80f7eaaba9488dba9af951a818e062             1  \n",
              "8ff3e24cb260420498b5537e372a1a1e             1  \n",
              "\n",
              "[5 rows x 25 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-6fb3679d-c802-4371-99c9-d201f8555606\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>pmcid</th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>mesh</th>\n",
              "      <th>keywords</th>\n",
              "      <th>authors</th>\n",
              "      <th>journal</th>\n",
              "      <th>pub_type</th>\n",
              "      <th>pub_date</th>\n",
              "      <th>...</th>\n",
              "      <th>pdf</th>\n",
              "      <th>xml</th>\n",
              "      <th>html</th>\n",
              "      <th>plain</th>\n",
              "      <th>pmc_tgz</th>\n",
              "      <th>xml_parse_d</th>\n",
              "      <th>html_parse_d</th>\n",
              "      <th>pdf_parse_d</th>\n",
              "      <th>plain_parse_d</th>\n",
              "      <th>content_text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>7575147222f842aba0727e3632c8f6dc</th>\n",
              "      <td>19902024</td>\n",
              "      <td>PMC2771255</td>\n",
              "      <td>Changes on the physiological lactonase activit...</td>\n",
              "      <td>Low caloric diet (LCD) is used for weight loss...</td>\n",
              "      <td>None</td>\n",
              "      <td>[BMI, high-density lipoprotein, low caloric di...</td>\n",
              "      <td>[Kotani K, Sakane N, Sano Y, Tsuzaki K, Matsuo...</td>\n",
              "      <td>Journal of clinical biochemistry and nutrition</td>\n",
              "      <td>[Journal Article]</td>\n",
              "      <td>2009-11-24</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>{'file_path': './output/formats/xmls/757514722...</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/757514722...</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8677a2c0e6214547acca432c72af98a2</th>\n",
              "      <td>28089709</td>\n",
              "      <td>PMC5357736</td>\n",
              "      <td>Objective assessment of dietary patterns by us...</td>\n",
              "      <td>BACKGROUND: Accurate monitoring of changes in ...</td>\n",
              "      <td>[Adult, Biomarkers/*urine, Cross-Over Studies,...</td>\n",
              "      <td>None</td>\n",
              "      <td>[Garcia-Perez I, Posma JM, Gibson R, Chambers ...</td>\n",
              "      <td>The lancet. Diabetes &amp; endocrinology</td>\n",
              "      <td>[Journal Article, Randomized Controlled Trial,...</td>\n",
              "      <td>2017-03-24</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/8677a2c0e...</td>\n",
              "      <td>{'file_path': './output/formats/txts/8677a2c0e...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>e937a9417a6e4723bdb6d07e4cd71a64</th>\n",
              "      <td>28601864</td>\n",
              "      <td>PMC5644969</td>\n",
              "      <td>Effect of a High-Protein Diet versus Standard-...</td>\n",
              "      <td>BACKGROUND: Some studies have shown that prote...</td>\n",
              "      <td>[Adult, Biomarkers/*blood, Blood Glucose/analy...</td>\n",
              "      <td>[Diet, Metabolic syndrome, Protein intake, Wei...</td>\n",
              "      <td>[Campos-Nonato I, Hernandez L, Barquera S]</td>\n",
              "      <td>Obesity facts</td>\n",
              "      <td>[Journal Article, Randomized Controlled Trial]</td>\n",
              "      <td>2017-11-24</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>0b80f7eaaba9488dba9af951a818e062</th>\n",
              "      <td>31138673</td>\n",
              "      <td>PMC6538848</td>\n",
              "      <td>A Multi-omics Approach to Unraveling the Micro...</td>\n",
              "      <td>Long-term consumption of dietary fiber is gene...</td>\n",
              "      <td>None</td>\n",
              "      <td>[AXOS, dietary fiber, glucose homeostasis, lip...</td>\n",
              "      <td>[Benitez-Paez A, Kjolbaek L, Gomez Del Pulgar ...</td>\n",
              "      <td>mSystems</td>\n",
              "      <td>[Journal Article]</td>\n",
              "      <td>2019-05-28</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>{'file_path': './output/formats/xmls/0b80f7eaa...</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/0b80f7eaa...</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8ff3e24cb260420498b5537e372a1a1e</th>\n",
              "      <td>31605240</td>\n",
              "      <td>PMC7165363</td>\n",
              "      <td>Elevated serum ceramides are linked with obesi...</td>\n",
              "      <td>INTRODUCTION: Low gut microbiome richness is a...</td>\n",
              "      <td>[Adult, Ceramides/*blood/metabolism, Chromatog...</td>\n",
              "      <td>[Ceramides, Endotoxin, Glucose metabolism, Mic...</td>\n",
              "      <td>[Kayser BD, Prifti E, Lhomme M, Belda E, Dao M...</td>\n",
              "      <td>Metabolomics : Official journal of the Metabol...</td>\n",
              "      <td>[Journal Article, Research Support, Non-U.S. G...</td>\n",
              "      <td>2019-10-11</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>{'file_path': './output/formats/xmls/8ff3e24cb...</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>5 rows √ó 25 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6fb3679d-c802-4371-99c9-d201f8555606')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-6fb3679d-c802-4371-99c9-d201f8555606 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-6fb3679d-c802-4371-99c9-d201f8555606');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-232eccea-0d0c-49c2-80ba-709613db1db3\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-232eccea-0d0c-49c2-80ba-709613db1db3')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-232eccea-0d0c-49c2-80ba-709613db1db3 button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "  <div id=\"id_82ceb078-3341-45a5-9e52-b2414cd78350\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('metadata_retrieved_df')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_82ceb078-3341-45a5-9e52-b2414cd78350 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('metadata_retrieved_df');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "metadata_retrieved_df"
            }
          },
          "metadata": {},
          "execution_count": 9
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tNyjMVOpRf24",
        "outputId": "59344c8d-280d-4266-9b10-08ca0531b6c2"
      },
      "source": [
        "metadata_retrieved_df.columns"
      ],
      "id": "tNyjMVOpRf24",
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Index(['pmid', 'pmcid', 'title', 'abstract', 'mesh', 'keywords', 'authors',\n",
              "       'journal', 'pub_type', 'pub_date', 'doi', 'issn', 'crossref',\n",
              "       'full_text_links', 'licenses', 'pdf', 'xml', 'html', 'plain', 'pmc_tgz',\n",
              "       'xml_parse_d', 'html_parse_d', 'pdf_parse_d', 'plain_parse_d',\n",
              "       'content_text'],\n",
              "      dtype='object')"
            ]
          },
          "metadata": {},
          "execution_count": 10
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "We can now call the 'parsed_to_df' function with the default value for the parameter 'path' to build the df using the same indexes and their coresponding full-texts."
      ],
      "metadata": {
        "id": "UXXZMepc4S_k"
      },
      "id": "UXXZMepc4S_k"
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df = parsed_to_df(path = './output/retrieved_parsed_files/content_text/')"
      ],
      "metadata": {
        "id": "uYtSGZ6q4rRz"
      },
      "id": "uYtSGZ6q4rRz",
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Only the lines where the full-text was retrieved are included.\n",
        "\n"
      ],
      "metadata": {
        "id": "o5hwq88z41x2"
      },
      "id": "o5hwq88z41x2"
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7yDxS9b74wdy",
        "outputId": "649fa368-f7d7-416c-d2c9-e18522f697b4"
      },
      "id": "7yDxS9b74wdy",
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(4, 1)"
            ]
          },
          "metadata": {},
          "execution_count": 12
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 175
        },
        "id": "HEldLL8I4y_C",
        "outputId": "9e93cdc9-08ca-4fd5-8dfe-c301933e8473"
      },
      "id": "HEldLL8I4y_C",
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                       content_text\n",
              "0b80f7eaaba9488dba9af951a818e062  Long-term consumption of dietary fiber is gene...\n",
              "7575147222f842aba0727e3632c8f6dc  Low caloric diet (LCD) is used for weight loss...\n",
              "8677a2c0e6214547acca432c72af98a2    S2213-8587(16)30419-3 Objective assessment o...\n",
              "8ff3e24cb260420498b5537e372a1a1e  INTRODUCTION: Low gut microbiome richness is a..."
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-3a0ef9ac-ff69-4410-a1d0-97019d4c17ea\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>content_text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0b80f7eaaba9488dba9af951a818e062</th>\n",
              "      <td>Long-term consumption of dietary fiber is gene...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7575147222f842aba0727e3632c8f6dc</th>\n",
              "      <td>Low caloric diet (LCD) is used for weight loss...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8677a2c0e6214547acca432c72af98a2</th>\n",
              "      <td>S2213-8587(16)30419-3 Objective assessment o...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8ff3e24cb260420498b5537e372a1a1e</th>\n",
              "      <td>INTRODUCTION: Low gut microbiome richness is a...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-3a0ef9ac-ff69-4410-a1d0-97019d4c17ea')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-3a0ef9ac-ff69-4410-a1d0-97019d4c17ea button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-3a0ef9ac-ff69-4410-a1d0-97019d4c17ea');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-0049fa4e-072d-4f66-9e45-b2fee2437e98\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0049fa4e-072d-4f66-9e45-b2fee2437e98')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-0049fa4e-072d-4f66-9e45-b2fee2437e98 button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "  <div id=\"id_9a6c9d6c-93e7-4dd4-9248-930116afca65\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('retrieved_df')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_9a6c9d6c-93e7-4dd4-9248-930116afca65 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('retrieved_df');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "retrieved_df",
              "summary": "{\n  \"name\": \"retrieved_df\",\n  \"rows\": 4,\n  \"fields\": [\n    {\n      \"column\": \"content_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 4,\n        \"samples\": [\n          \"Low caloric diet (LCD) is used for weight loss. Paraoxonase 1 (PON-1) is associated with the antioxidant functions of high-density lipoprotein (HDL). Among limited data on the relationships between obesity and PON-1, there has been no study on the effects of a stand-alone LCD on the physiological lactonase activity of PON-1. We investigated the prospective effects of LCD intervention (2 months) for weight loss on serum PON-1 activities (lactonase, arylesterase [mono-esterase] and tri-esterase) and HDL cholesterol (HDL-C), and their association with low-density lipoprotein cholesterol (LDL-C) in overweight and non-morbidly obese but otherwise healthy women (n = 30; mean age, years; mean body mass index [BMI], kg/m(2)). In addition to the data such as BMI, blood pressure, blood glucose and lipids, PON-1 activities were examined between pre- and post-intervention. The intervention reduced all metabolic outcomes, and PON-1 lactonase activity (determined with 5-[thiobutyl]butyrolactone) significantly decreased by , paralleled by arylesterase (by ) and tri-esterase (by ). In multiple regression analysis, the percent change of PON-1 lactonase was significantly, positively and independently correlated to that of LDL-C (beta = ), HDL-C (beta = ), and BMI (beta = ). Our results showed that the solo diet treatment on weight loss might reduce serum PON-1 lactonase activity with reduced HDL-C and LDL-C. The relationship between the lactonase and LDL-C may be adaptive, plausibly hypothesizing less need for PON-1 activity as an antioxidant property to protect lipoproteins. Further research is needed to confirm this prediction. Obesity is a crucial risk factor in the development of its co-morbid diseases such as metabolic syndrome and atherosclerotic disorder including cardiovascular disease (CVD), and the increased prevalence of obesity is a major problem in many countries []. Impaired high-density lipoprotein (HDL) antioxidant defenses in obese people, such as a decreased activity of the HDL-associated enzyme, paraoxonase 1 (PON-1), have recently received increased attention []. PON-1, an esterase carried by HDL, is known to exert a protective effect against oxidative damage of cells and lipoproteins, playing an anti-inflammatory and anti-atherogenic role [\\u2013]. In fact, a low circulating PON-1 activity has been reportedly associated with obesity, metabolic syndrome and CVD [, \\u2013]. Low caloric diet (LCD) is one of the treatment modalities adopted as a part of lifestyle management on weight loss, leading to the improvement of metabolic and cardiovascular conditions [, ]. A few studies have addressed the changes in circulating PON-1 activity in the treatment intervention for obesity [\\u2013]. These previous results are not always consistent. In one, weight loss by gastric banding significantly increased PON-1 activity in morbidly obese patients []. In another study with obese patients treated with orlistat (an anti-obesity drug), PON-1 activity was significantly increased []. Weight reduction via a combination of exercise and diet energy restriction significantly reduced PON-1 []. As mentioned above, HDL is widely recognized to be an anti-atherogenic lipoprotein. Whereas the influence of calorie restriction diet for weight loss on lipid profiles, circulating HDL cholesterol (HDL-C) in particular, is reportedly controversial, HDL-C has been known to be often reduced in short-term diets [\\u2013]. The clinical significance and detailed mechanism of this is presently unclear, so accumulating data are required. Turnover studies have documented the changes in clearance of HDL-C during dietary modification [], and the reduction in chylomicron-derived HDL formation following caloric (lipid) restriction diet is discussed as an alternate explanation []. There is an opinion that the diet-induced HDL-C drop does not increase CVD risk []; however, whether this diet-induced reduction of HDL-C is beneficial/harmful remains debatable. One of the physiological roles of HDL involves the action of PON-1 [, ], and to evaluate a functional aspect of HDL in measuring PON-1 activity may yield further insights on this issue. The presence of dysfunctional HDL may, at the same HDL-C levels, signify quite different outcomes, and the same may be said about inflammatory/anti-inflammatory HDL. PON-1 hydrolyzes lipid peroxides, imparting protection to low-density lipoprotein (LDL) against oxidation [, ]. Although conflicting results exist [], a low PON-1 activity generally implies their reduced ability to protect LDL from oxidation, resulting in increased oxidized LDL []. There is a positive association between oxidized LDL and LDL cholesterol (LDL-C) concentrations [], and a LCD treatment may have the beneficial effects of reducing LDL-C levels. Therefore, when we assess the effects of LCD on PON-1, its association with the beneficial changes of LDL-C is an important factor to consider, that is one aim of the present study. Moreover, the previous studies to observe the association between weight loss and PON-1 had two main limitations. First, none has addressed the effects of LCD as a stand-alone treatment on PON-1 activity. Interventions by a combination of exercise and diet, or diet and orlistat, could make it difficult to determine which aspect of the intervention is responsible for the change of PON-1 (or HDL-C) and to what extent. In particular, when a diet treatment is combined with exercise, HDL-C metabolism can be modified [, ] by the improvement of skeletal muscle/liver and insulin homeostasis through exercise [, ]. One of our study aims was thus to investigate the effects on PON-1 of an intervention by stand-alone \\u2018LCD\\u2019 to generate results that are easier to interpret and accordingly more meaningful in practice. The second limitation of the previous studies is that they have employed substrates that can not necessarily reflect the physiological activity of PON-1, which is recently regarded primarily as a lactonase [, ]. Hence, measuring the lactonase would provide news in research on the diet-PON-1 association. With this background in mind, the present study investigated, in a population of overweight and not-morbidly obese but otherwise healthy subjects, the prospective effects of a solo LCD intervention for weight loss on the physiological lactonase activity of serum PON-1. We also addressed its association with changes in LDL-C, HDL-C and other clinical outcomes. Taking into account the gender difference in HDL-C metabolism as a powerful confounder [, ], our work was restricted only to a female cohort. In total, 30 healthy and free-living Japanese women, aged 33 to 63 (mean age,  +/- ) years, with an overweight/obesity level of body mass index (BMI) (mean,  +/- kg/m; range, to ) participated in this study. Exclusion criteria were as follows: the use of medications and nutrient supplements, pregnancy, current smoking, alcohol abuse, psychological contraindications as determined by study investigators, or known hypersensitivity to any of the ingredients of the formula. The study was approved by the Ethics committee of Kyoto Medical Center, and all subjects gave informed consent. Concerning the LCD prescription, during the study intervention period, subjects were placed on the LCD (5023 kJ/day) (consisting of protein, 47%; lipid, 7%; carbohydrate, 40%) with meal replacement (Diet\\u2019s, Suntory Co. Ltd., Osaka, Japan) every dinner for 2 months. The subjects were also instructed by trained dieticians about the planning of nutritionally balanced diets, and the adherence was checked through dietary daily records. They were encouraged not to change usual physical activity during the intervention. The subjects\\u2019 data at baseline (pre-intervention) and the end of 2 months\\u2019 study (post-intervention) were used. In an overnight fast, we confirmed the subjects\\u2019 data on BMI, blood pressure (measured using a mercury sphygmomanometer), plasma glucose (enzymatic kinetic method, Shino-test Co. Ltd., Tokyo, Japan) and lipids such as total cholesterol (enzymatic kinetic method, Wako Pure Chemical Industries Co. Ltd., Tokyo, Japan) and triglyceride (enzymatic kinetic method, Daiichi Pure Chemicals Co. Ltd., Tokyo, Japan). Both HDL-C and LDL-C were measured using a homogeneous assay (Daiichi Pure Chemicals Co. Ltd.). Serum PON-1 lactonase activity was kinetically determined at 37 degreesC using 5-(thiobutyl)butyrolactone (TBBL), a lactone substrate that is analogous to \\u03b3-nonanoic lactone, yet releases upon hydrolysis of the \\u03b3-butyrolactone ring, and recorded at 405 nm in a VERSA max (Tunable) Microplate Reader [, ]. Serum PON-1 tri-esterase activity was determined using paraoxon as a substrate, from the initial velocity of p-nitrophenol production at 37 degreesC, and recorded at 405 nm []. Serum PON-1 arylesterase (mono-esterase) activity was determined using phenylacetate as a substrate and kinetically measured at 37 degreesC by recording absorbance changes at 270 nm [, ]. Data are expressed as mean +/- standard deviation (SD). The level change between pre- and post-intervention in respective variables was tested by paired t test. To detect the correlations between the percent change of PON-1, HDL-C and LDL-C, we used Pearson\\u2019s rank coefficient test as well as multiple regression analysis controlling for the above variables and age. A p< was considered significant. All subjects completed the present LCD intervention. The data on respective variables in pre- and post-intervention were listed in Table . During the intervention, most variables (except for systolic blood pressure which did not achieve a statistical significance level) were significantly reduced. Notably, PON-1 lactonase, as well as arylesterase and tri-esterase activities, were significantly reduced. As listed in Table , in the simple correlation tests, the percent change of BMI or LDL-C was significantly and positively correlated to that of PON-1 lactonase activity, along with age. Multiple regression analysis on PON-1 lactonase activity, adjusted for age, BMI, LDL-C and HDL-C, revealed that the percent change of LDL-C, HDL-C, and BMI (the correlation degree ranked in this order) was significantly, positively, and independently correlated to that of lactonase activity. Furthermore, multiple regression analysis on the percent change of HDL-C, adjusted with age, BMI, LDL-C and PON-1, revealed that the percent change of BMI was significantly, inversely, and independently correlated, and PON-1 lactonase activity was significantly, positively, and independently correlated to HDL-C. Additionally, the results in the correlation tests on arylesterase or tri-esterase activity were similar with those in PON-1 lactonase activity (data not shown). Our study displays the effects of short-term LCD intervention for weight loss on PON-1 physiological lactonase activity and HDL-C, and their association to LDL-C changes, in healthy overweight/obese women. So far, evidence on the influence of weight loss on PON-1 has been inconsistent [\\u2013], and this may be partly explained by the differences in intervention methods and populations. In our cohort, all measured variables including BMI, HDL-C and PON-1 were reduced, and the reduction of BMI showed a significant positive correlation to that of PON-1. A previous study has reported that a combination of exercise and diet could reduce PON-1 in subjects with the metabolic syndrome [], though in this case, the element of exercise may make some differences in the HDL metabolism from a solo diet treatment [, ]. Another previous study has also implicated that a healthy diet, for instance focused on vegetables, might reduce PON-1, though this study was not necessarily for obese subjects []. Taken together, our results seem to raise the possibility that a diet treatment reduces PON-1. Whether the mechanism was due to metabolic changes of HDL-C [, ] or a reaction to diet- and weight loss-induced reduction of oxidation (as described later) remains to be determined presently, but at least, our study adds to the present knowledge in two main areas: 1) the results from a better defined cohort of a homogeneous population of healthy and overweight/obese (non-morbidly) individuals and a well-controlled intervention by di one, 2) the novelty of demonstrating a decrease in lactonase activity that parallels the mono-esterase and tri-esterase changes. PON-1 catalyzes the hydrolysis of numerous substrates: lactones, thiolactones, esters and phospho-triesters, including paraoxon, from which its name is derived. Nevertheless, only after many years of research, it has became apparent that PON-1 is actually a lactonase, catalyzing the hydrolysis and synthesis of several lactones [, , ]. The other activities, such as arylesterase and phospho-triesterase, might be simply promiscuous, although they are used in clinical laboratory research as a practical way of measuring PON-1 activities (seen in the previous articles on weight loss) and even the commercial kits have appeared on the market this year. The physiological substrates of PON-1 remain uncertain; however, they are presumably lactones in food components or derivatives of fatty acid oxidation chain reactions such as 5-hydroxyeicosatetraenoic acid lactone. These end-product lactones of fatty acid oxidation may be found in LDL, HDL or macrophages [, ]. Based on the current wisdom that PON-1 is a protective factor against CVD [\\u2013], the lowering of such PON-1 activities might be interpreted as a detrimental effect of the diet. However, we think that this has to be balanced by the beneficial effects produced by the diet on other well-known risk factors for CVD. In this regard, our study has found an association that may explain this apparent \\u2018paradoxical\\u2019 effect of the LCD intervention. Indeed, to the best of our knowledge, this is the first description that there is a significant relationship between the lowering of PON-1 activity (its lactonase or the others) and that of LDL-C. Namely, the correlation between PON-1 and LDL-C changes in the LCD intervention has not been previously reported, although whether this is a specific effect of LCD is unclear. Interestingly, from our study results, this relationship was not fully explained by the reduction in HDL-C. Shedding more light on the mechanism for the relationship between PON-1 and LDL-C is a future challenge, but we would suggest this hypothetical flow of events. PON-1 hydrolyzes lipid hydroperoxides and lactones in oxidized lipoproteins, imparting protection to LDL against oxidation [, , , , , ]. LDL oxidation could be related to LDL-C levels [] and/or to obesity [, ]. Weight loss and caloric restriction diet induces the prominent reduction of the LDL oxidation [, ]. Our study demonstrated a weak correlation between the decrease levels of BMI and those of LDL-C (r = , p> [data not shown in the result section]), as well as an independently significant correlation of the decrease levels of PON-1 to those of BMI or LDL-C (shown in Table ). Such a weak correlation between obesity and LDL-C (a quantity measure of LDL), relative to the measures of LDL oxidation (a concept of quality of LDL), has been noted previously [, ]. Thus, our results on the relationship between PON-1 and LDL-C might be explained, a priori, as a consequence of less need for LDL protection by the LDL-C concentration decrease and weight loss following a LCD intervention. Importantly, in the last years, the re-evaluation of HDL physiology stresses its pro- or anti-inflammatory functions according to different circumstances and its pleiotropic functional activities [, ]. Hence, the decrease in PON-1 as well as HDL-C in response to LCD may not be detrimental but a physiologically appropriate or adaptive phenomenon. Whether this is due to adjustments in the expression of PON-1 or to re-distribution of HDL particles needs to be addressed in further studies. As a study limitation, we examined a restricted number of women only. More studies on a larger scale of subjects, including men, and other ethnic populations (may have different genetic factors) are needed to establish our results [, ]. Also, studies evaluating PON-1 mass changes are warranted. Even though we did not measure actual PON-1 mass, the fact that all three PON-1 activities measured are reduced in a similar manner suggests that the changes are likely due to changes in PON-1 concentrations. Since the correlation with changes in HDL-C is not so strong, this may indicate lower amounts of PON-1 per unit HDL particle. In a cohort of healthy overweight/obese women, we showed for the first time a reduced PON-1 lactonase activity (as well as its mono- and tri-esterase activities) during a LCD intervention on weight loss, with a significant correlation with the reduction of LDL-C. The present results on the relationship between PON-1 and LDL-C may be pointing to the PON-1 activity level needed to protect LDL from oxidation, as an adaptive response to the LDL-C decrease and weight loss by the diet treatment. Moreover, the apparently paradoxical changes in HDL-C and PON-1 lactonase activity seen in the diet on weight loss would emphasize the need for the clinical laboratory to explore lipoprotein metabolisms beyond the simple static HDL-C measurement. Further research is needed to confirm our theory.\",\n          \"INTRODUCTION: Low gut microbiome richness is associated with dyslipidemia and insulin resistance, and ceramides and other sphingolipids are implicated in the development of diabetes. OBJECTIVES: Determine whether circulating sphingolipids, particularly ceramides, are associated with alterations in the gut microbiome among obese patients with increased diabetes risk. METHODS: This was a cross-sectional and longitudinal retrospective analysis of a dietary/weight loss intervention. Fasted serum was collected from 49 participants (41 women) and analyzed by HPLC-MS/MS to quantify 45 sphingolipids. Shotgun metagenomic sequencing of stool was performed to profile the gut microbiome. RESULTS: Confirming the link to deteriorated glucose homeostasis, serum ceramides were positively correlated with fasting glucose, but inversely correlated with fasting and OGTT-derived measures of insulin sensitivity and beta-cell function. Significant associations with gut dysbiosis were demonstrated, with SM and ceramides being inversely correlated with gene richness. Ceramides with fatty acid chain lengths of 20-24 carbons were the most associated with low richness. Diet-induced weight loss, which improved gene richness, decreased most sphingolipids. Thirty-one MGS, mostly corresponding to unidentified bacteria species, were inversely correlated with ceramides, including a number of Bifidobacterium and Methanobrevibacter smithii. Higher ceramide levels were also associated with increased metagenomic modules for lipopolysaccharide synthesis and flagellan synthesis, two pathogen-associated molecular patterns, and decreased enrichment of genes involved in methanogenesis and bile acid metabolism. CONCLUSION: This study identifies an association between gut microbiota richness, ceramides, and diabetes risk in overweight/obese humans, and suggests that the gut microbiota may contribute to dysregulation of lipid metabolism in metabolic disorders. Type 2 diabetes is the result of insulin resistance and deterioration of \\u03b2-cell function. Yet the environmental factors, and their links to physiological mediators, that drive the development of type 2 diabetes are only partially understood. The gut microbiota has emerged as a potential contributor to the regulation of glucose homeostasis. Type 2 diabetes is associated with alterations in a number of bacteria and their biological functions in both Chinese () and European () populations. Metabolic changes that precede frank diabetes can also be transferred via the gut microbiota to mice (). Low gut microbiota richness, which is altered in some moderate obesity/overweight patients without overt diabetes (), may be an important factor in the progression to diabetes. The microbial ecosystem can be characterized by its richness . the number of unique bacterial genes present in a metagenome. People in Western societies, with high rates of obesity, type 2 diabetes, and frequent exposure to antibiotics and food additives that can impact gut microbes, have considerably lower bacterial diversity than previously unexposed hunter-gatherers (). Gastrointestinal diseases such as liver cirrhosis () and inflammatory bowel diseases () have been associated with a loss of metagenomic richness. Metabolic characteristics are also altered in individuals with low gut microbiome richness, as Danish subjects with fewer microbial genes have increased BMI, insulin resistance, inflammatory markers, as well as hypertriglyceridemia (). These associations were also observed in French individuals and our laboratory further showed that this was partially reverses by diet-induced weight loss (). Lipid metabolism is critical to the pathogenesis of diabetes () and alterations in the gut microbiota could potentially regulate these pathways. Sphingolipids are a diverse class of membrane lipids that also act as intracellular signaling molecules. Ceramides, which accumulate with lipid loading and inflammation, are increased in patients with type 2 diabetes (; ), and they can directly induce insulin resistance () and regulate \\u03b2-cell apoptosis (). Plasma ceramides may even become clinical biomarkers, but the environmental factors that contribute to ceramide accumulation, outside of obesity, are not well defined (). It was recently shown in mice, however, that the gut microbiota regulates liver ceramide synthesis and subsequent hepatic steatosis and gluconeogenesis through FXR-dependent secondary bile acid metabolism (). Furthermore, endotoxin-induced inflammation, a potentially important link between impaired glucose homeostasis and gut dysbiosis (), readily stimulates sphingolipid synthesis. It is still unknown, however, whether an altered gut microbiota is associated with increased ceramide levels in humans. Combining targeted lipidomics and quantitative metagenomics, the objective of the present study was to examine whether ceramide accumulation could link gut dysbiosis to increased diabetes risk in individuals with overweight or obesity. The study was conducted at Pitie\\u0301-Salpe\\u0302trie\\u0300re Hospital (Nutrition department) in Paris, France. Forty-nine participants were overweight (N=11) and obese (N=38) (8 males, 41 females), and have been described in detail elsewhere (; ). Participants did not have diabetes, chronic or inflammatory diseases, or related treatments, and did not take antibiotics within the two months prior to stool collection. The intervention consisted of a 6-week period on a low calorie diet enriched with soluble fibers, high protein intake, and low glycemic index carbohydrates. Daily caloric intake was approximately 1,200 kcal/d for women and 1,500 kcal/d for men, composed of 35% protein, 25% fat and 40% carbohydrate. A 6-week weight stabilization diet followed the calorie restriction period, and was established based on each subject\\u2019s resting energy expenditure and physical activity levels. Details of the analyses have been described elsewhere (; ). Body composition was determined using dual energy x-ray absorptiometry for the estimation of total body fat and android fat proportion. Subcutaneous adipose tissue samples were evaluated for adipocyte size and immune cell frequency (). Blood samples were collected after an overnight fast to measure routine biochemical parameters. Serum insulin was assayed by a chemiluminescense (ARCHITECT system, Abbott). Insulin sensitivity was quantified using the following fasting indices: the Homeostasis Model Assessment of insulin sensitivity (HOMA-S), calculated using the HOMA2Calculator system (); Quantitative Insulin Sensitivity Check Index (QUICKI) (); and the Disse index (), a lipid-based index of insulin sensitivity An oral glucose tolerance test (OGTT) was performed in 40 subjects (39 that had insulin measured during the test at all time points). The Matsuda Index was calculated as a measure of insulin sensitivity (). \\u03b2-cell function was estimated by the insulin secretion and sensitivity index-2 (ISSI-2), an OGTT-analog of the IVGTT-derived disposition index. It is the product of insulin secretion in response to glucose, measured as the ratio of insulin AUC to the glucose AUC, and the Matsuda index. The ISSI-2 captures the hyperbolic relationship between insulin sensitivity and secretion () and correlates with the disposition index (). We confirmed the hyperbolic relationship in our sample population, as outlined in the original paper (data not shown) (). Analysis of sphingolipids was conducted by HPLC-MS/MS, as described previously (), in overnight fasted serum. Lipids were extracted in acidified methanol:chloroform with both internal and calibration standards (Avanti Polar Lipids, Alabaster, AL, USA). Forty-five sphingolipids were quantified, namely sphingomyelins (SM), dihydroceramides (DHCer), defined as Cer(d18:0) in standard nomenclature, and sphingosine- or sphingadienine-containing ceramides, that is Cer(d18:1) and Cer(d18:2), respectively. These 45 lipids were detected in at least 80% of samples. To avoid bias introduced by simple 0-imputation of the values below detection, measurements below the level of quantitation were imputed using multiplicative log-normal-randomized imputation with the zCompositions package in R. Total faecal DNA was extracted and sequenced on a high-throughput ABI SOLiD sequencer. Reads were cleaned for quality and origin (human, cow, and plant), mapped to the million gene reference catalogue () and normalized by nucleotide length. Forty-five subjects with sufficient sampling depth were used for subsequent analyses. Gene richness was calculated, after downsizing to 7 million genes, as the total number of uniquely mapped genes per sample. Richness categories were split at 460,000 genes in the catalog, as done previously (). Metagenomic species (MGS) were constructed from total-sum normalized gene abundance data using the catalog provided by Nielsen  and only co-abundance clusters greater than 700 genes were considered as bacterial species (). Computations were performed using the [momr] R package (; ) Kegg Ortholog (KO) annotation was provided by the reference catalog, and each gene was summed into its respective KO. Annotation of bile acid genes was performed as described by others (). Briefly, protein sequences from organisms containing complete bile acid pathways were extracted, specifically: Clostridium scindens ATCC 35704, Bifidobacterium longum SBT2928, Bacteroides fragilis NCTC9343, Eggerthella lenta DSM 2243, and Ruminococcus gnavus ATCC 29149. Sequences were then aligned to the reference catalog using UBLAST with an E-value cutoff of 10. These candidates were then aligned against the KEGG database to include only genes with best hits to bile acid proteins. Bile acid genes were grouped by their functional annotation in MetaCyc (. bile acid degradation). Statistical analyses were performed in the R Language and Environment with packages listed in [brackets]. As the data were generally positively skewed (lipidomics) or non-normal (metagenomics), robust statistics were used when possible. The Spearman method was used for correlations. The MM-type robust regression [lmrob function in robustbase] was used for multiple regression (). Longitudinal data were analyzed using mixed effects models with random intercepts for subject [lme4, car, multcomp]. Enrichment of KEGG modules and pathways were tested with the reporter features algorithm using adjusted p-values and coefficients from the correlations of each KO to Cer(d18:1) [piano] (). Only distinctly up- and down-regulated gene sets are reported. The null distribution was used for the significant assessment of gene sets between 3 and 100 KOs. For clinical (including gene richness) and lipid class associations, unadjusted p-values were reported. However, given the a priori nature of metagenomic analysis, p-values for tests of MGS or KOs was adjusted for the Benjamini-Hochberg false discovery rate; the only exception being bile acid gene set enrichment, as this hypothesis was pre-specified (). Characteristics of the population used for lipidomics profiling have been described previously (), and are summarized in and and . Median BMI was approximately 33 kg/m and body fat was 40%. None of the subjects had diabetes. Shotgun metagenomic analysis indicated that 41 women had a median gene richness of 507,800 and 8 men had a median gene richness of 537,000, which were not statistically significantly different. We confirmed that gene richness was inversely associated with fasting TG (r=\\u2212, P<), as previously reported. Serum sphingolipids were tested for associations with a number of diabetes risk factors. Four sphingolipid subclasses were quantified, covering a total number of 45 lipids including sphingomyelins (SM), dihydroceramides (DHCer) or Cer(d18:0) in standard nomenclature, and (d18:1)Cer or (d18:2)Cer ceramides, which correspond to a classical sphingosine backbone or a monounsaturated variation, respectively. The most significant associations found were with the non-SM subclasses and clinical variables linked to deteriorated metabolic homeostasis such as accumulation of android fat, age, cholesterol, triglycerides, and GGT (). Importantly, regarding glucose homeostasis, all subclasses were positively associated with 60 min glucose during the OGTT, but only two subclasses, DHCer and Cer(d18:1), were associated with impaired insulin sensitivity determined by both fasting (Disse, QUICKI) and the OGTT-derived Matsuda index (). Furthermore DHCer, Cer(d18:1), and Cer(d18:2), inversely associated with the ISSI-2 index, indicative of impaired \\u03b2-cell function (). SM, Cer(d18:1) and Cer(d18:2) were also positively correlated with circulating CRP or IL-6. In order evaluate the relative weight of each sphingolipid subclass in the association with clinical parameters, the sum of the number of variables (excluding cholesterol and TG) with significant associations with individual shphingolipid species was calculated. This revealed that DHCer and Cer(d18:1) were more strongly associated with clinical parameters than other species; together they accounted for 7 of the top 9 lipids with the highest number of non-lipid (. HDL-C) clinical correlations (; ). Importantly, very-long chain ceramides (C22 \\u2013 C24 species), which are the most abundant species in human serum (), were as related to clinical phenotype as C16 and C18 species. Overall, these data confirm that serum ceramides levels are associated to a deleterious metabolic profile. Sphingolipids may be an important component of the dyslipidemia associated with low bacterial richness. Consistent with this hypothesis, there was an inverse association between gene richness and each lipid subclass, although the statistical significance was strongest for Cer(d18:1) (rho=\\u2212, P=) and Cer(d18:2) (rho=\\u2212, P=) (, ). In multivariable analysis adjusting for age, sex, and fasting triglycerides, SM was no longer significant at P<, while Cer(d18:1), Cer(d18:2), and to a lesser extent DHCer, remained statistically significant (). At the level of individual sphingolipids species, the relationships between SM species were generally attenuated compared to the Cer(d18:1) and Cer(d18:2) (). After adjustment, the strongest associations were among species containing between 20 and 24 carbons in Cer(d18:1) or Cer(d18:2) (). These data demonstrate a significant link between serum ceramides and gut microbiota richness, and reveal unexpected importance of long chain ceramides in this association. Longitudinal changes in gene richness would bolster the cross-sectional findings. As reported previously (), diet-induced weight loss significantly increased gene richness in subjects with low gene richness before the intervention (). We therefore examined if weight loss also corresponded to changes in circulating sphingolipids. DHCer, Cer(d18:2), Cer(d18:1), and SM were decreased by 47%, 39%, 35%, and 26%, respectively, after a 6-week period of diet restriction in the obese subjects (all P<), and remained suppressed\\u2014with some rebound\\u2014during the 6 additional weeks of weight stabilization (). Over the weight loss period, few changes were seen in patients with high gene richness at baseline, whereas whose with initial low gene counts ameliorated their low microbiome diversity (). Regression of lagged variables confirmed a longitudinal association: an increase in gene richness from the previous time point was associated with a concomitant decrease in sphingolipid subclasses among participants with low baseline richness (). Overall, these data demonstrate association between blood ceramide levels and dynamic gut microbiota status driven by diet interventions in subjects with overweight/obesity. We next investigated if some specific bacterial species could be related to host ceramide levels. Lower gene richness was associated with a decrease in 97 of the 310 MGS (Padj< ), consistent with the loss of bacterial richness. Firmicutes, the most abundant phylum in the human intestine, comprised the majority of associations with both gene richness and sphingolipids (). Decreased abundance of 30 MGS were associated with an increase serum concentrations of Cer(d18:1); approximately half of these bacteria (14 MGS) also showed significant association with Cer(d18:2) in the same direction (). A number of the sphingolipid-related MGS could be annotated to the species level. In particular, Methanobrevibacter smithii, Bifidobacterium adolescentis, Alderceutzia equolifaciens, Ruminococcus obeum, and Eubacteria hallii were associated with both low gene richness and high serum levels of Cer(d18:1). Alistipes indistinctus, Dialister invisus, and Bifidobacterium bifidum were also negatively associated with Cer(d18:1), but not with gene richness. Only one MGS was inversely associated with DHCer and one with SM. Ruminococcus gnavus was positively associated with Cer(d18:1). Thus from this analysis, it appears that the link between serum ceramide levels and gut microbe communities mainly relate to the loss of bacterial subgroups. Quantitative metagenomics allows direct characterization of the functional potential of the microbiota. Consistent with the MGS data, gene richness was associated with changes in a large number of orthologue genes grouped within KO annotations (approximately half of total KOs detected). Interestingly, serum levels of Cer(d18:1) and Cer(d18:2) showed significant association with approximately 20% of annotated KO, and only a small percentage related to DHCer and no KOs significantly correlated to SM (). Ninety-one percent of Cer(d18:1) and 95% of Cer(d18:2) associated KOs were also associated with gene richness (). Because studies in mice have uncovered secondary bile acid metabolism by the gut microbiota as an important regulator of ceramides (), we examined the relationship between Cer(d18:1) and bacterial genes involved in bile acid transformation. Genes involved in glycoholate metabolism, bai genes coding the 7\\u03b1-dehydroxylation pathway for secondary bile acid synthesis (bile acid degradation), and iso-bile acid synthesis genes were found with a decreased frequency with higher host serum ceramides levels (Padj<), which suggest different potential for bile acid transformation in subjects with elevated ceramides. Analysis at the module level confirmed the inverse association with bile acid metabolism genes (p=), and bile acid degradation (P=) more specifically (). Longitudinal data confirmed these findings since patients with low richness at baseline exhibited an increase in abundance of several bile acid genes, including bsh, during the dietary intervention concomitant with the increase in richness and decrease in ceramides (). A hypothesis-free analysis of all available KEGG modules and pathways was also performed at baseline. At the module level (), the strongest inverse association with serum Cer(d18:1) were related to methanogenesis and cellular processes comprised of archaea-specific enzymes (RNA polymerase and archaeal proteasome). Lower capacity for nutrient transport including branched-chain amino acids and polar amino acids were a notable signature bacterial communities with elevated host Cer(d18:1). At the pathway level (), lower abundance of general cellular processes, metabolism of alanine, and carotenoid biosynthesis were detected. The majority of KEGG modules enriched with higher Cer(d18:1) were related to biosynthesis. Lipopolysaccharide biosynthesis was highly enriched at the module level, as was the biosynthesis of amino acids (tryptophan) and B-vitamins (thiamine, pantothenate). Higher capacity for LPS biosynthesis and flagellar assembly were positively related to Cer(d18:1) at the pathway level. Thus, differences in overall metagenomic richness, bacterial abundance, and functional capacity are related to host ceramides accumulation. We herein establish an association between the human gut microbiota and ceramides\\u2014lipid mediators implicated in the development of type 2 diabetes. After confirming the links between serum ceramides and diabetes risk, we found that low metagenomic richness was associated with significantly higher levels of serum ceramides. Decreased abundance of Methanobreviibacter smithii and anti-inflammatory bifidobacteria species were related to higher ceramides levels. Importantly, the ceramide-associated microbiome demonstrated an increase in a number of biosynthetic pathways, notably increased LPS synthesis and flagellar assembly. Thus, low metagenomic richness may contribute to increased diabetes risk in humans by permitting an inflammatory dysbiosis that induces ceramide accumulation. Studies in mice have implicated the gut microbiota in the regulation of host metabolism (). Gut microbiota composition is also altered in patients with type 2 diabetes from both China and Europe (; ). In the Chinese population, the individuals with type 2 diabetes had an increase in a number of pathogenic bacteria and a decrease in anti-inflammatory bacteria (). In Swedish individuals, type 2 diabetes and elevated triglycerides were primarily associated with decreased abundance of bacterial species, and their metagenomes were enriched in fructose and mannose metabolism and phosphotransferase systems, similar to our findings in subjects without type 2 diabetes. More recently, it was shown that metformin treatment at least partially confounds the relationship with type 2 diabetes (), therefore a major strength of the current study is the persistence of these findings in untreated participants. Decreased gut microbial richness is observed in overweight and obese patients, and is associated with markers of insulin resistance, systemic inflammation (. CRP), and hypertriglyceridemia (), and these factors are improved with diet-induced weight loss (). Low gene richness has also been associated with more severe conditions such as ulcerative colitis (). Thus, microbiota richness is lower with both metainflammation and chronic inflammatory diseases, which may stimulate ceramide production. In addition to lipid loading, toll-like receptor activation leads to the accumulation of ceramides (). Lipopolysaccharide, the major TLR-4 ligand, induces both de novo ceramide synthesis () and conversion of SM into ceramides via sphingomyelinases (). Endotoxin-induced cytokines also lead to ceramide accumulation (). LPS can induce ceramide accumulation in muscle and liver (), as well as lead to increased secretion of ceramides into lipoprotein particles and serum (). Chronic endotoxemia and cytokine signaling could therefore induce ceramide accumulation in obesity. Indeed, high fat feeding in mice increases circulating LPS two- to three-fold and increases the proportion of LPS-containing bacteria in the gut (). Conversely, prebiotic treatment that restores bifidobacterium species in the intestine of rodents are correlated with reduced endotoxemia and improved glucose homeostasis (). We observed a decrease in Bifidobacterium bifidum and Bifidobacerium adolescentis with higher ceramide levels, consistent with these observations in mice. Notably, Ruminococcus gnavus was positively associated with ceramides, and this species is increased in inflammatory bowel disease and appears to regulate the composition of the mucosa-associated bacterial network in these diseases (). The positive associations between increased LPS biosynthesis and flagellar assembly, another pathogen-associated molecular pattern (), corroborates the link between ceramides and gut-derived inflammation. Together, albeit only observational, our results support the hypothesis that the loss of overall diversity permits a shift towards increased endotoxin burden in the intestine, activating inflammatory pathways in the host, . ceramide accumulation. Microbial bile acid metabolism may also regulate host ceramides synthesis. Jiang  showed that the gut microbiota regulates serum ceramide levels through intestinal farsenoid-X-receptor (FXR) signaling (). Specifically, they found that reduced hydrolysis of tauro-beta-muricholic acid (by bsh), an antagonist of intestinal FXR signaling, reduces intestinal FXR activation, mitochondrial dysfunction, and circulating ceramides. In contrast, we did not detect a relationship between bsh gene abundance, and even found that reduced capacity for overall secondary bile acid metabolism was associated with higher circulating ceramides. The design of our study prohibits conclusive statements regarding the role of secondary bile acid metabolism and ceramides, but future studies incorporating careful analysis of fecal and serum bile acid levels, as well as bsh activity in stool, could help clarify this potentially important mechanism. Regarding different ceramide species, we found that longer chain Cer(d18:1), namely C20, C22, and C24 species, were the most strongly associated with both metabolic parameters and gene richness. This acyl distribution, regulated by the CERS2 gene, suggests that the gut microbiota may be particularly related to hepatic ceramide synthesis (). These results are contrary to recent genetic studies in mice that show that Cer(d18:1/16:0) is the primary driver of metabolic dysfunction (), and other reports of no effect () or even a beneficial effect of very-long chain ceramides (). Similar conclusions were also drawn from an epidemiological study of coronary artery disease (). On the other hand, nearly all circulating ceramides, regardless of fatty acids, have approximately the same degree of association with type 2 diabetes and insulin resistance (), or, as with our study, show equivalent or greater elevations in C18 to C24 ceramides compared to Cer(d18:1/16:0) (; ). Consistent with the relationships to \\u03b2-cell function in our study, circulating very-long chain DHCer predict incident type 2 diabetes (). These discrepant findings may be reconciled by a specific action of circulating ceramides. LDL-enriched in Cer(d18:1/16:0) or Cer(d18:1/24:0), the latter being far more abundant in serum, induce insulin resistance equivalently in vitro and in vivo (). Further, extracellular ceramides, derived from exogenous delivery, can enhance hepatic gluconeogenesis, consistent with the positive association with fasting glucose in our study (). Further work is therefore needed to determine the causal role, if any, of circulating ceramides, and their acyl-chain length, in the pathogenesis of insulin resistance in humans. Several limitations of our study should be noted. Most of the evidence for ceramide-induced insulin resistance comes from cellular accumulation within metabolic tissues. While we have discussed the evidence for a potential role for circulating ceramides, future studies that include both metagenomic analysis and tissue lipidomics would be an important next step. Our study is observational: while we have adjusted for a number of potentially important confounders, larger human studies and gut microbiota transfer studies would both aid in confirming the importance of the gut microbiota on ceramide metabolism. In conclusion, we confirmed that serum ceramides are positively correlated with markers of both insulin resistance and impaired \\u03b2-cell function. Inflammation is an important driver of ceramide synthesis, and our data suggest that gut dysbiosis, characterized by decreased metagenomic richness, lower abundance of anti-inflammatory bifidobacteria species, and a greater metagenomic capacity for toll-like receptor ligand synthesis, may contribute to this pathway in human obesity. This study paves the way for future experiments aimed at elucidating the role of the gut microbiota in host lipid metabolism and the pathogenesis of type 2 diabetes.\",\n          \"Long-term consumption of dietary fiber is generally considered beneficial for weight management and metabolic health, but the results of interventions vary greatly depending on the type of dietary fibers involved. This study provides a comprehensive evaluation of the effects of a specific dietary fiber consisting of a wheat-bran extract enriched in arabinoxylan-oligosaccharides (AXOS) in a human intervention trial. An integrated multi-omics analysis has been carried out to evaluate the effects of an intervention trial with an AXOS-enriched diet in overweight individuals with indices of metabolic syndrome. Microbiome analyses were performed by shotgun DNA sequencing in feces; in-depth metabolomics using nuclear magnetic resonance in fecal, urine, and plasma samples; and massive lipid profiling using mass spectrometry in fecal and serum/plasma samples. In addition to their bifidogenic effect, we observed that AXOS boost the proportion of Prevotella species. Metagenome analysis showed increases in the presence of bacterial genes involved in vitamin/cofactor production, glycan metabolism, and neurotransmitter biosynthesis as a result of AXOS intake. Furthermore, lipidomics analysis revealed reductions in plasma ceramide levels. Finally, we observed associations between Prevotella abundance and short-chain fatty acids (SCFAs) and succinate concentration in feces and identified a potential protective role of Eubacterium rectale against metabolic disease given that its abundance was positively associated with plasma phosphatidylcholine levels, thus hypothetically reducing bioavailability of choline for methylamine biosynthesis. The metagenomics, lipidomics, and metabolomics data integration indicates that sustained consumption of AXOS orchestrates a wide variety of changes in the gut microbiome and the host metabolism that collectively would impact on glucose homeostasis. (This study has been registered at under identifier NCT02215343)IMPORTANCE The use of dietary fiber food supplementation as a strategy to reduce the burden of diet-related diseases is a matter of study given its cost-effectiveness and the positive results demonstrated in clinical trials. This multi-omics assessment, on different biological samples of overweight subjects with signs of metabolic syndrome, sheds light on the early and less evident effects of short-term AXOS intake on intestinal microbiota and host metabolism. We observed a deep influence of AXOS on gut microbiota beyond their recognized bifidogenic effect by boosting concomitantly a wide diversity of butyrate producers and Prevotella copri, a microbial species abundant in non-Westernized populations with traditional lifestyle and diets enriched in fresh unprocessed foods. A comprehensive evaluation of hundreds of metabolites unveiled new benefits of the AXOS intake, such as reducing the plasma ceramide levels. Globally, we observed that multiple effects of AXOS consumption seem to converge in reversing the glucose homeostasis impairment. The World Health Organization (WHO) reports that billion adults are overweight, 650 million of whom are obese (), making the obesity epidemic and the resulting metabolic complications an important health concern. The current rise in obesity prevalence in low- and middle-income countries, particularly in urban settings, and in children and adolescents further underlines the importance of prevention and treatment of obesity (, ). Obesity is the result of a long-term imbalance between energy intake and expenditure, mainly caused by overnutrition and sedentary lifestyle (). Obesity is characterized by chronic low-grade inflammation and impairment of the lipid and glucose metabolism, thus increasing the risk of developing comorbidities such as type 2 diabetes (T2D), cardiovascular disease (CVD), and mental diseases (, ). Different epidemiological studies support the notion that regular consumption of enriched dietary fiber food is inversely correlated with weight gain, thus potentially reducing the risk of developing T2D and CVD (). Interestingly, a recent umbrella review of systematic reviews reports that dietary fiber intake has a convincingly protective effect against CVD, including coronary artery disease and CVD-related death, and that there is suggestive evidence of disease risk reduction for several type of cancers, T2D, and stroke (). Accordingly, dietary fiber consumption appears to be a feasible long-term and cost-effective strategy to prevent obesity and its comorbidities. Dietary fiber comprises a diverse group of structurally complex carbohydrates with various effects on human metabolism and the gut microbiota (). Arabinoxylans (AX) are cell wall components that constitute a major part of the dietary fiber fraction of cereal grains and are an important fiber source in the human diet (). The main products of AX enzymatic hydrolysis are arabinoxylan-oligosaccharides (AXOS) and xylan-oligosaccharides (XOS), which show prebiotic potential because of their ability to increase the abundance of bifidobacteria and some butyrate producers in the gut (). The bifidogenic effect of AX-derived oligosaccharides has been shown in human dietary interventions where abundance of Bifidobacterium species has been increased in feces as a consequence of the intake of 4-g/day XOS for 3 weeks () or 2- to 10-g/day AXOS during a similar period of time (). Although no major impact on blood biochemical, physiological, and anthropometrical parameters has been observed following AXOS/XOS intake, most likely due to the short duration of the studies, the increases in fecal moisture () and short-chain fatty acid (SCFA) concentrations (), and in postprandial ferulic acid concentration (), altogether confirm utilization of this type of dietary fiber by gut microbiota and its potential impact on metabolic health. In a previous study including a randomized crossover intervention evaluating the effects of AXOS and polyunsaturated fatty acids (PUFA) in overweight subjects with indices of metabolic syndrome (MetS), we reported that AXOS exert a bifidogenic effect and increase the abundance of butyrate-producing bacteria using a 16S rRNA gene sequencing approach (). In the current paper, we aimed to further characterize the microbiome, metabolome, and lipidome responses to AXOS intake to gain insight into the possible role of the microbiota as mediator of dietary effects on metabolic health (). To this end, we have analyzed the biological samples of the group of responders to AXOS consumption (who showed significant changes in their gut microbiota) from our previous study (Kj\\u00f8lb\\u00e6k  []). A multi-omics approach has been applied, including shotgun DNA sequencing metagenomics of feces; nuclear magnetic resonance (NMR) metabolomics conducted in feces, plasma, and urine; and mass spectrometry-based (liquid chromatography mass spectrometry [LC-MS]) lipidomics in plasma and feces. The new taxonomic analyses performed on metagenomic shotgun sequencing data indicate that AXOS intake increased the abundance of Actinobacteria (P <= ), Bifidobacteriaceae (P <= ), and Bifidobacterium (P < ) taxonomy groups, comparing samples at baseline with those after the intervention. In addition to this characteristic bifidogenic effect, the abundance of members of the Ruminococcus gnavus group (P <= ) and of the Lachnospiraceae XPB1014 group (P <= ) were also increased in samples after the AXOS intervention. Conversely, the AXOS intake reduced the proportion of Rikenella (P <= ), Parabacteroides (P <= ), and Paraprevotella (P <= ) species (). The metagenome analysis was based on the processing of more than billion reads (106 million paired-end reads on average per sample), which produced approximately million encoding genes assembled. A nonredundant coding metagene database was obtained by clustering protein sequences predicted from sequencing reads, thus retrieving \\u223c million metagenes as a reference for the assessment of the effects of AXOS intake on the gut microbiome. We found that 3,230 metagenes (\\u223c of the full data set) exhibited differential abundance, of which 852 were underrepresented and 2,378 were overrepresented after the intervention. In order to confidently disclose the taxonomy categories corresponding to those genes, we performed a simple BLASTP () search against the nonredundant protein database at the NCBI (release May 2017), making taxonomy assignments of query sequences with alignments of at least 70% amino acid sequence identity, compared with the respective top matches. Regarding the underrepresented set of metagenes, we were able to assign taxonomy categories for 36% of them with certainty and identify species of the genera Bacteroides, Alistipes, Eubacterium, Roseburia, Prevotella, and Raoultella (). Of all the species, the metagenes of Bacteroides species were those most negatively influenced by AXOS intake, particularly Bacteroides massiliensis and Bacteroides eggerthii. Additionally, species such as Alistipes obesi, Eubacterium siraeum, Roseburia inulinivorans, Prevotella buccalis, and Prevotella multisaccharivorax were also reduced by AXOS intake. The analysis of the overrepresented set of metagenes (60% with reliable taxonomy identification) allow the identification of Bifidobacterium species, with Bifidobacterium adolescentis, Bifidobacterium longum, Bifidobacterium catenulatum, and Bifidobacterium angulatum as the most predominant (). We observed that more than one-third of the overrepresented metagenes corresponded to Prevotella species, particularly Prevotella copri (). We detected and quantified DNA reads for the six Prevotella phylotypes present in the SILVA database release 128 (see in the supplemental material). Interestingly, each individual phylotype showed no significant distribution, including the \\u201cPrevotella_9\\u201d corresponding to P. copri, but with a notable increase in reads of \\u201cPrevotella_1\\u201d and \\u201cPrevotella_9.\\u201d Globally, the sum of reads for all Prevotella phylotypes indicated that such species are promoted by AXOS intake as well (P < ). The analysis of the overrepresented set of metagenes also enabled the identification of additional bacterial species, such as Bacteroides stercoris, Bacteroides ovatus, Blautia obeum, Blautia wexlerae, Eubacterium rectale, Eubacterium hallii, Roseburia faecis, and Adlercreutzia equolifaciens, which were increased after AXOS intervention (). (ii) Functional analysis. The set of metagenes underrepresented as a consequence of AXOS intake exhibits a wide range of metabolic and cellular functions according to the annotation with KEGG. Of these, some metagenes encode potential pathogenic features, such as biofilm formation (ko:02026, ko:02025, and ko:05111), beta-lactam resistance (ko:01501), bacterial secretion systems for virulence factor release (ko:03070), flagellar assembly (ko:02040), cationic antimicrobial peptides (CAMP resistance) (ko:01503), bacterial chemotaxis (ko:02030), monobactam biosynthesis (ko:00261), and lipopolysaccharide biosynthesis (ko:00540) (). When we analyzed the overrepresented set of metagenes, we found a lower level of functional annotation than for the underrepresented set of metagenes (24% and 43%, respectively). We then evaluated the function of those metagenes present in the species predominantly influenced by AXOS, such as Prevotella copri and Bifidobacterium spp., which account for roughly 50% of the overrepresented metagenes (). As shown in , the AXOS consumption led to an expansion of the collection of genes dedicated to vitamin and cofactor biosynthesis as well as of genes specialized in transport, biosynthesis, and degradation of glycans. Notably, we also found a significant increase of genes involved in the biosynthesis of the neurotransmitter gamma-aminobutyric acid (GABA) and precursors of aromatic amino acids (shikimate and chorismate) with potential roles in the gut-brain communication. Moreover, the Venn analysis indicated that the functions amplified to a greater extent by AXOS intake (present in at least 3 out of the 4 groups of overrepresented metagenes) consisted of those involved in the biosynthesis of vitamin K2, the derivative vitamin B9 tetrahydrofolate (THF), vitamin B2, and the coenzyme A biosynthesis. All the above were equally found to be overrepresented in metagenes from Prevotella copri and Bifidobacterium species (). To investigate the bacterial cellular functions boosted by AXOS consumption, we conducted a domain enrichment analysis using the Pfam annotation system. This showed that the overrepresented metagenes were almost exclusively dedicated to glycan metabolism, with several metagenes exhibiting a wide range of particular functions for this fundamental cell process (). We observed overrepresentation of functions such as outer membrane proteins (OMPs) for carbohydrate binding and transport relying on OMP-, TonB-, and SusD-like proteins. Moreover, we also identified a vast amount of proteins enriched in carbohydrate binding modules (CBMs) characteristic of glycosyl hydrolase enzymes such as beta-galactosidases, of which central domains were likewise enriched in the overrepresented metagenes. More-general functions were also found in several metagenes that encode ATPases with no defined molecular roles but apparently related to sugar transport function according to associations inferred from functional and phylogenetic information. Increases in the presence of metagenes encoding carboxypeptidases and protein domains associated with similar functions (PD40/WD40-like) suggested that AXOS could increase the activity of specific peptidases of the gut metagenome. An increased presence of several metagenes associated with the prokaryote innate immune response, such as restriction modification systems, was also observed (). (iii) Human gut metavirome response to AXOS intake. We also used the metagenome information to evaluate the presence of virus sequences in the raw data retrieved from shotgun sequencing of fecal DNA. By using the ViromeScan approach (), we were able to evaluate the relative abundance of approximately 22 virus families. In general terms, we found a predominant tendency toward a decrease of reads matching human virus sequences after the AXOS intervention, although the differences did not reach statistical significance. Interestingly, our viral characterization on the gut samples led to the detection of Megavirales and other giant viruses (>200 nm in diameter), which were only recently identified in human stool and other human samples (). The clinical or biological significance of the presence of these viruses in the human gut remains to be determined; for this reason, our data pave the way to searching for giant viruses in the human gut and to establishing the impact that diet has on them. AXOS impact on markers of glucose homeostasis and gut microbiota. A new evaluation in the set of responders indicates that AXOS intake did not result in significant changes in most of the metabolic health outcomes and biomarkers analyzed (anthropometry, inflammatory, lipid, or glucose biomarkers), probably due to the limited sample size and short study duration. Nevertheless, we observed that the mean value of fasting insulin as well as of the insulin resistance index (homeostatic model assessment for insulin resistance [HOMA-IR]) was slightly reduced (, , and ). Although this reduction was observed in only a proportion of the subjects analyzed (\\u223c47%), it is suggestive of a potential improvement in glucose metabolism probably linked to changes in certain microbiota signatures. Using a logistic regression analysis, we found that changes in the HOMA-IR tended to be related to three different taxonomy groups, whose abundance was increased as a result of the intervention (). In particular, the changes in abundance of \\u201cLachnospiraceae AC2044 group\\u201d (odds ratio = , confidence interval [CI] = to , P = ), \\u201cEubacterium ruminantium group\\u201d (odds ratio = , CI = to , P = ), and \\u201cLachnospiraceae XPB1014 group\\u201d (odds ratio = , CI = to , P = ) suggest that an increase of 1 logarithmic unit in the relative abundance of these microbial groups leads to an increase of at least 3-fold in the probability of the HOMA-IR amelioration. By using MS-based lipidomics approaches, we were able to quantify 9 lipid classes and more than 150 species according to the chain length and double bond content in both plasma and fecal samples. We found no significant changes in the lipid profiles from feces in response to the intervention. In plasma, we detected a significant increase in one out of 18 analyzed cholesteryl ester species, CE 20:0, and a decrease in two hexosylceramides and one ceramide (). For the latter, a quasiconsensus signal of reduction in plasma ceramide levels was observed in 8 out of the 9 features evaluated (three of them in a significant manner, P < ), thus obtaining a global trend for a decrease in the total ceramide content in plasma samples (P <= ). We found no correlations between microbiome signatures and this lipidomic feature. The NMR-based metabolomic analysis of the metabolite profile before and after the AXOS intervention indicated that there were changes only in the urine hippuric acid concentration (). This increase was more prominent in a subgroup of participants, which could be attributed to higher abundance of Prevotella_6-associated species (RY = , QY = ) and negatively correlated with Ruminococcaceae abundance (RY = , QY = ). Additional correlation patterns suggested that urine dimethylamine (DMA) levels were associated with proportions of Peptoclostridium species (RY = , QY = ). In fecal samples, positive correlations between Prevotella_2 abundance and the concentrations of acetate, propionate, and succinate were identified (RY = , QY = ). Finally, application of a supervised model of analysis allowed partial discrimination between plasma samples before and after the intervention (start and end points). Signals from residual very-low-density lipoprotein (VLDL) and triglyceride (TAG) fractions, including fatty acids and glyceryls of lipids, were to some extent increased after the intervention. Although this could be indicative of a trend for dyslipidemia, the VLDL profile in plasma measured by classical biochemical methods indicates no major changes in this type of lipid during the intervention (P <= ). On the other hand, the abundance of Eubacterium rectale, a microbial species boosted by AXOS intake () (), was positively correlated with concentrations of phosphocholine and sn-glycero-3-phosphocholine in plasma (RY = , QY = ). Using a multi-omics approach, we have performed an in-depth characterization of the microbiome, lipidome, and metabolome response to a specific source of fiber (AXOS). In previous intervention trials, there has been controversy on the effect of AXOS on health outcomes with the microbiome assessments being limited to taxonomic features (, , , , ). The present study aimed at providing a more exhaustive analysis of the AXOS-driven metabolic changes with a particular focus on the early gut microbiota-induced changes after a 4-week randomized dietary intervention. In agreement with previous findings, we found that a bifidogenic effect of AXOS was evident when the metagenome was analyzed using shotgun sequencing (, ). In this study, we report new microbiome effects resulting from AXOS intake, including increases in the abundance of Ruminococcus gnavus and Lachnospiraceae XPB1014 taxonomy groups, the latter related to Eubacterium, Blautia, and Roseburia species (BLASTN search against the nonredundant 16S NCBI database, 91 to 94% sequence identity). We have described the AXOS impact on Blautia luti and Blautia wexlerae species and other butyrate/propionate-producing bacteria such as Eubacterium rectale, Eubacterium hallii, Faecalibacterium prausnitzii, and Dorea longicatena, based on V3-V4 16S amplicon sequencing data and the combination of multivariate and linear analyses of operational taxonomic units (OTUs) in a previous paper (). Moreover, we observed that AXOS intake significantly reduced Parabacteroides, Paraprevotella, and Rikenella species. Similarly, reductions in Parabacteroides proportions have been linked to improvement of glucose homeostasis in obese mice after dietary supplementation with capsaicin according to another study (). This effect produced by AXOS on Paraprevotella species was also observed after the Roux-En-Y gastric bypass (RYGB)-like and metformin treatments in the Zucker diabetic fatty (ZDF) rat model of obesity and T2D (). Upon assembling and predicting the set of bacterial genes present in the gut microbiome of responders included in the present study, we also found that AXOS intake increased the proportion of Prevotella species and particularly of Prevotella copri. When searching the SILVA database (release 128), we found that of the six different phylotypes present in the database, only two tend to be increased after AXOS intake (Prevotella_1 and Prevotella_9). However, a change in this taxonomy category (P < ) was detected as a result of the intervention when all Prevotella phylotypes were combined and considered in the analysis. These findings could lead to questioning of the proper identification of this complex group of bacteria. This issue has been reported previously in a human study where De Filippis and coworkers describe that correlation patterns of Bacteroides and Prevotella species with animal- or plant-derived nutrients, respectively, correspond with a simplistic association given the unexplored genetic variability at the subgenus level of these taxonomic groups prevalent in the human gut microbiota (). The role of the species P. copri in the amelioration of glucose intolerance seems to be contradictory according to the existing scientific evidence. While the early metagenomic assessment of the gut microbiome of T2D patients indicated that P. copri is linked to an increased production of branched-chain amino acids (in plasma and gut) associated with insulin resistance and to a higher risk of T2D (), other animal studies reported that P. copri has a positive impact on glucose homeostasis, given the activation of the intestinal gluconeogenesis via succinate production (). Therefore, it is difficult to attribute a healthy or harmful role to P. copri in glucose homeostasis in the light of the existing evidence. We hypothesize that such antagonist effects could depend on the specific strains tested, on the substrate availability (activating different proteolytic or saccharolytic pathways), and on other components of the gut microbiota involved in host-microbe and microbe-microbe interactions and on the experimental conditions. In fact, a very recent work of De Filippis  suggests that antagonist associations of P. copri strains with health and disease would be explained by genetic variability and certain metabolic traits (). In our particular case, AXOS intake increased a large number of metagenes carried by P. copri and tended to improve the HOMA-IR index (P = ), suggesting that AXOS intake can ameliorate the glucose metabolism, via modification of metagenomic functions, in overweight individuals with signs of MetS, in agreement with other studies in humans (, ). Nevertheless, this slight improvement in HOMA-IR values was not explained by a change in abundance of any Prevotella phylotype, including those closely related to P. copri. In contrast, we unveiled an association trend between the HOMA-IR and three different taxonomy groups, including the Lachnospiraceae XPB1014 group related to Eubacterium, Blautia, and Roseburia species. Special attention should be given to the species from the Lachnospiraceae XPB1014 group since these were found to be augmented by AXOS intake (). It is likely that several components of the microbiota involved in the generation of SCFAs by cross-feeding mechanisms could explain potential AXOS effects on glucose metabolism. Such effects could be mediated, for example, by the conversion of succinate into SCFAs like propionate, which can trigger intestinal gluconeogenesis and improve glucose metabolism (). From a functional point of view, we observed that AXOS consumption increased the abundance of genes involved in carbohydrate metabolism as well as vitamin and cofactor biosynthesis. The expansion of the set of glycosyl hydrolases and enzymes related to carbohydrate metabolism, essentially in P. copri and Bifidobacterium species, could be involved in the generation of organic acids, including SCFAs, through cross-feeding mechanisms with potential beneficial effects on gut and metabolic health (). Theoretically, microbiome-mediated increases in the production of vitamins and cofactors could also bring host benefits (., vitamin K2 in bone physiology altered in obese individuals [, ]), although direct evidence of vitamin production and host utilization remains to be shown. The AXOS intake increased the representation of metagenes involved in the biosynthesis of tetrahydrofolate (THF; the active form of folic acid), which may also have potential metabolic benefits. Indeed, some obese individuals show folate deficiency, which could lead to hyperhomocysteinemia related to atherosclerosis risk (). In line with this, administration of folic acid and derivatives seems to confer protection against hyperhomocysteinemia-induced oxidative stress (). AXOS intervention also increased the capacity of the gut microbiome for aromatic amino acid production (shikimate pathway producing precursors for tryptophan, phenylalanine, and tyrosine biosynthesis), which theoretically could increase the generation of derivative neurotransmitters (., serotonin, dopamine, epinephrine, and norepinephrine). An augmented capacity for GABA biosynthesis was also detected as a consequence of AXOS intake. Altogether, this could result in additional effects on the communication within the gut-brain axis (). Different studies based on mouse models indicate that low concentrations of GABA are associated with anxiety-like behavior, insulin resistance, energy expenditure imbalance, and weight gain in overweight and obesity (). Particularly, one animal study indicated that GABA acts as activator of insulin secretion, given that GABA-like immunoreactive cells in the pancreas of rats are reduced in diabetes (). A recent study in obese rats also indicated that GABA supplementation reduces serum ceramide concentrations (). Therefore, we hypothesized that the AXOS-driven increased abundance of microbial species encoding GABA production could hold promise for the mediation of metabolic and mental health effects. Interestingly, other microorganisms like Bacteroides species, evolutionarily related to the Prevotella genus members, as well as some Lactobacillus and Bifidobacterium species, also have been described to be GABA-producing gut microbes (, ). Our findings also show that AXOS intake significantly decreases the concentration of several ceramide species in plasma (P < ). Previous reports have also shown an association between the increased levels of ceramides and impaired glucose homeostasis in MetS and T2D patients (). Finally, the metabolomics approach was useful to trace the utilization of AXOS by gut microbiota through the presence in urine of hippuric acid, a microbial metabolite derived from degradation of polyphenols associated with dietary fiber (). Our data suggest that hippurate production could be related to the abundance of Prevotella species increased by AXOS intake. Additionally, acetate, propionate, and succinate concentrations in feces were also positively related to the abundance of Prevotella species, which could bring benefits to metabolic health as explained above (). The plasma metabolomic profiling indicates that phosphatidylcholine biosynthesis could be increased by AXOS intake via the butyrate producer Eubacterium rectale, which showed positive correlations with phosphatidylcholine precursors (P < ). In turn, this could decrease bioavailability of choline for the production of methylamines (DMA and trimethylamine [TMA]) by other gut microbes (), such as Peptoclostridium species that showed a positive correlation with the DMA urine concentration in our study. TMA is the precursor of trimethylamine oxide (TMAO), which is produced in the liver, kidney, and other tissues. Increased plasma levels of TMAO have been associated with enhanced risk of developing atherosclerosis, T2D, and chronic kidney disease. Therefore, this could be a mechanism by which AXOS microbiome-mediated effects could confer protection against the impairment of glucose metabolism, (), as well as against a wide variety of chronic diseases (), reducing the availability and exposure to methylamine oxide forms. Indeed, a previous study in patients with chronic kidney diseases found that AXOS slightly decreased plasma TMAO (), results that further support our ideas. Using a multi-omics functional approach, we have characterized in depth the effects of AXOS intake on the microbiome, lipidome, and metabolome and, thereby, tentatively identified possible microbiome- and non-microbiome-mediated dietary effects on metabolic health. We have shown that in addition to the well-recognized bifidogenic effect of AXOS, this type of dietary fiber increases the abundance of Prevotella and clostridial bacteria from the Lachnospiraceae family (butyrate producers) along with increases in organic acids (propionate and succinate). Furthermore, AXOS decreases plasma ceramide levels via a microbiome-independent mechanism, which altogether could contribute to improved glucose metabolism. The direction of other functional metagenomic changes induced by AXOS was related to the production of neuroactive (GABA) and choline metabolites, suggesting potential additional effects that could reduce the risk of developing chronic metabolic conditions. Studies with a larger sample size and longer duration are warranted to confirm whether the metabolome and lipidome profiles resulting from AXOS intake and the induced microbiome configuration are translated into metabolic health outcomes, aiding in the validation of the biomarkers of dietary exposure and function tentatively identified in the present study. The study is registered at (), was conducted according to the guidelines laid down in the Declaration of Helsinki, and was carried out in accordance with the ethical standards of the responsible regional committee on human experimentation in Denmark, registered as H-4-2014-052, and the Danish Data Protection Agency (2013-54-0522). The present study is based on a non-placebo-controlled randomized crossover trial with two dietary intervention periods (4 weeks each) separated by a washout period (4 weeks) conducted in 30 overweight and obese individuals (body mass index [BMI] of 25 to 40 kg/m) with MetS index (an increased waist circumference plus at least one of the criteria for MetS []) described in detail elsewhere (). The dietary fiber ingredient provided was a wheat bran extract, enriched in arabinoxylan-oligosaccharides (AXOS) ( g/day AXOS). Before and after intervention periods, weighted dietary records and fecal, urine, and blood samples as well as anthropometric measurements were collected (). The present study encompasses the integration of clinical and multi-omics data derived from multiple biological samples of a subset of responders to AXOS intervention (n = 15) (), for whom AXOS did produce meaningful shifts in the structure of the gut microbial community (with no risk of carryover effect because of the crossover design). The information inferred from beta-diversity analysis, based on the Bray-Curtis dissimilarity index among samples (PERMANOVA = , P < ), was used as an indicator of significant changes in the microbial community structure (). The baseline characteristics of this cohort of samples are described in in the supplemental material. Responders were defined as the group of participants included in the same dietary intervention period and showing significant changes in their microbiota composition, for whom the OTUs matching with bifidobacteria and/or recognized butyrate producer species were boosted as a result of the dietary intervention with AXOS and assessed by16S rRNA gene sequencing (). Collection of fecal and urine samples took place at home prior to the clinical investigation day (CID) in sterile collection tubes provided. The urine sample was collected in the morning, and the fecal sample was collected as close to the urine sample as possible. Both samples were kept at 4 degreesC after collection and delivered to the Department of Nutrition, Exercise, and Sports, University of Copenhagen, within 3 h after collection of feces. At the Department, the samples were weighed and the density of the urine sample was measured (Atago PAL-10S pocket refractometer) to calculate the volume. For the metabolomic analysis of urine, an aliquot of 4 ml urine was used and 45 \\u03bcl sodium azide was added. For fecal analysis, an aliquot of the sample was transferred to the EasySampler kit for stool collection (GP Medical Devices, Denmark) for metagenomic analyses. Here, the fecal sample was homogenized with MilliQ water, 1:1, and an aliquot was used for metabolomic analyses. For lipidomic analysis, 500 \\u03bcl methanol was added per 500 mg homogenized fecal sample. All aliquots were stored at \\u221280 degreesC. (ii) Blood samples. Prior to the CID, the participants consumed a standardized dinner in the evening followed by a fasting period of at least 8 h. All blood samples were collected in the fasting state. Lipidomic analyses were conducted in plasma and serum. For plasma analysis, blood was collected in EDTA tubes and put directly on ice, and for serum analysis, blood was kept at room temperature for 20 min to coagulate before centrifugation. For metabolomic analysis, blood for plasma analyses was collected in heparin tubes and put directly on ice. Afterwards, all blood samples were centrifuged at 2,500 x g for 10 min at 4 degreesC to obtain respective aliquots of serum and plasma and stored at \\u221280 degreesC. Sampling procedures for plasma and serum samples presented here\\u2014for example, glucose, insulin, lipids, inflammation markers, etc.\\u2014are found elsewhere (). The fecal DNA was extracted using the QIAamp Fast DNA stool minikit (Qiagen, Hilden, Germany) according to the manufacturer\\u2019s instructions with a prior step of bead beating in 2-ml microcentrifuge tubes containing glass beads, \\u223c200 mg feces, and 1 ml InhibitEX buffer. Bead beating was carried out in a Mini-Bead Beater apparatus (BioSpec Products, Bartlesville, OK, USA) with two cycles of shaking during 1 min and incubation on ice between cycles. The fecal DNA was quantified through Qubit and the Qubit dsDNA HS assay kit (Thermo Fisher Scientific, Waltham, MA, USA), and \\u03bcg DNA of every sample was sent to be multiplexed and sequenced in a plate of the HiSeq2500 platform with 2 x 125 paired-end configuration (Eurofins Genomics GmbH, Ebersberg, Germany). The NEBNext Ultra DNA library prep kit for Illumina was used according to the manufacturer\\u2019s instructions with 400- to 500-bp insert size and low PCR cycling (5 cycles). (ii) Metagenome data analysis. Approximately Tb raw data were delivered in fastq files. Paired-end fastq files were used to assemble the fecal metagenome of each individual at two different time points by using Velvet assembler v () with k-mer length 61, -exp_cov auto, and -ins_length 200 parameters, followed by an assembly refinement step using the Metavelvet extension () with the -ins_length 200 -ins_length_sd 50 configuration. The assembled contigs larger than 200 nucleotides (nt) in length were retained, and the prediction of potential open reading frames (ORFs) contained in such fragments from respective metagenomes was assisted by FragGeneScan (), with the -complete=0 and -train=complete configuration. Peptide sequences obtained from the ORF prediction in all metagenomes were concatenated and clustered at 70% sequence identity using cdhit algorithm with -c , -G 1, -M 10,000, -B 1, and -g 1 parameters (, ). For read mapping against the nonredundant peptide database compiled from the 30 metagenomes assembled, we used the Usearch algorithm with the following parameters: -usearch_local, -id , -strand both, and -maxaccepts 1. Differential abundance of coding metagenes was assessed by using negative binomial distribution methods implemented in edgeR () and determining a false-discovery rate (FDR) for selection of <. For taxonomy aims, we mapped the remaining set of reads with no hits after comparison against the nonredundant coding database. Consequently, we mapped those reads against the reference Silva database (release 128, ), and read alignments were filtered to retain those expanding beyond 80% of the read length (>=100 nt) with >=99% sequence identity. Differential abundance in phylum, family, and genus distribution was evaluated with Linear Mixed Models (LMM) by using time points as fixed effects and subject-specific information (gender, age, and BMI) as random effects. Additionally, we assessed the taxonomy distribution of the coding metagenes with differential abundance by using BLAST and the NCBI nonredundant protein database (). Taxonomy identification for those metagenes was based on selection of hits with the best alignment score among the multiple alignments (covering 100% query sequence and at least 70% sequence identity) of each query against targets retrieved. Cases were defined as \\u201cuncertain\\u201d when equal scoring was obtained in comparison with more than one hit belonging to different microbial species. (iii) Functional analyses. Preliminary analysis to assess the function of differentially abundant metagenes was completed by submitting the amino acid sequence of metagenes with significant increasing and decreasing abundance, as a consequence of the AXOS intake, to the Kyoto Encyclopedia of Genes and Genomes (KEGG) Automatic Annotation Server (KAAS) (). Advanced functional enrichment analysis was performed by annotating the full set of nonredundant coding sequences, obtained in the global metagenome assembly, against the Pfam database () through the WebMGA server (). Functional enrichment of Pfam functions in samples after AXOS intake was evaluated by hypergeometric Fisher's exact test with correction for multiple testing using the Benjamini-Hochberg method. (iv) Biochemical and lipidomic data. Plasma biochemical markers previously assessed () were reanalyzed in the present study for the selected subset of subjects. The Shapiro-Wilk normality test was estimated for all variables, and accordingly, paired and one-sided t test or Wilcox signed-rank test was performed, respectively, in order to establish if the AXOS intervention improved any of the biochemical parameters measured. To establish possible relationships between changes in the abundance of bacterial genera and markers of glucose metabolism (HOMA-IR and fasting insulin), a logistic regression model was applied, using the glm [family=\\u201cbinomial\\u201d(link=\\u201clogit\\u201d)] function of R the HOMA-IR index was used as binary outcome (1 = improved and 0 = nonimproved), and the changes in the relative abundance of different bacterial genera (\\u0394genus = log10 normalized reads at endpoint \\u2212 log10 normalized reads at start point) were used as explanatory variable. Lipidomics data were analyzed using Wilcox signed-rank test for paired samples with Benjamini-Hochberg correction for multiple comparisons when appropriate. (v) Human virome analysis. Raw fastq sequences were processed using the ViromeScan software to taxonomically characterize the virome directly from metagenomic reads () using the -d human_DNA parameter to detect only DNA viruses with humans as a natural host. The relative abundance was used to perform statistical analyses and comparisons among samples before and after the AXOS intervention. Alpha-diversity was computed for each sample considering the number of viral species detected within each metagenome. Significance testing was performed using the R package stats and nonparametric Wilcox signed-rank test for paired samples. When appropriate, P values were adjusted for multiple comparisons using the Benjamini-Hochberg correction. A false-discovery rate (FDR) of < was considered statistically significant. All the statistics and plots for metagenomics and functional approaches were obtained and designed on R . Lipids were quantified by direct flow injection electrospray ionization tandem mass spectrometry (ESI-MS/MS) in positive ion mode using the analytical setup and strategy described previously (, ). Lipid extraction was performed according to the method of Bligh and Dyer (), in the presence of non-naturally occurring lipid species as internal standards. The following lipid species were added as internal standards: phosphatidylcholine (PC) 14:0/14:0, PC 22:0/22:0, phosphatidylethanolamine (PE) 14:0/14:0, PE 20:0/20:0 (diphytanoyl), phosphatidylinositol (PI) 17:0/17:0, lysophosphatidylcholine (LPC) 13:0, LPC 19:0, sphingosine-based ceramides (Cer) d18:1/14:0, Cer 17:0, D7-free cholesterol (FC), cholesteryl ester (CE) 17:0, and CE 22:0. A fragment ion of m/z 184 was used for PC, sphingomyelin (SM) (), and LPC (). Neutral loss fragments were used for the following lipid classes: PE and PI with a loss of 141 and 277, respectively (, ). PE-based plasmalogens (PE P) were analyzed according to the principles described by Zemski Berry and Murphy (). Cer and hexosylceramides (HexCer) were analyzed using a fragment ion of m/z 264 (). Free cholesterol (FC) and CE were quantified using a fragment ion of m/z 369 after selective derivatization of FC (). Quantification was achieved using two non-naturally occurring internal standards (IS) for each lipid class (except for PI, sphingomyelin [SM] was calculated using PC IS and PE P were calculated using PE IS), and calibration lines were generated by standard addition of a number of naturally occurring species to plasma. Calibration lines were generated for the following naturally occurring species: PC 34:1, 36:2, 38:4, and 40:0 and PC O-16:0/20:4; SM d18:1/16:0, 18:1, and 18:0; LPC 16:0, 18:1, and 18:0; PE 34:1, 36:2, 38:4, and 40:6 and PE P 16:0/20:4; Cer d18:1/16:0, 18:0, 20:0, 24:1, and 24:0; FC, CE 16:0, 18:2, 18:1, and 18:0. Deisotoping and data analysis for all lipid classes were performed by self-programmed Excel macros as described previously (, ). Lipid species were annotated according to the recently published proposal for shorthand notation of lipid structures that are derived from mass spectrometry (). Glycerophospholipid species annotation was based on the assumption of even-numbered carbon chains only. SM species annotation is based on the assumption that a sphingoid base with two hydroxyl groups is present. (ii) Quantification of bile acids. Both plasma and fecal bile acids were quantified by LC-MS/MS using stable isotope dilution analysis. Fecal samples were homogenized as described below using 10-fold dilution of the samples. (iii) Quantification of fecal lipid species. Fecal samples were homogenized in 70% 2-propanol, using a gentleMACS dissociator (Miltenyi Biotec GmbH, Bergisch Gladbach, Germany) as described previously (). Fecal sterols and 5\\u03b1/\\u03b2-stanols were quantified by liquid chromatography\\u2013high-resolution mass spectrometry (LC-MS/HRMS) after derivatization to N,N-dimethylglycine esters (). Fecal fatty acids were quantified by gas chromatography coupled to mass spectrometry (GC-MS) after preparation of fatty acid methyl esters (). In order to reduce the water signal, urine samples (1 ml) were freeze-dried and reconstituted in 650 \\u03bcl NMR phosphate buffer [sodium phosphate, M, pH sodium 3-(trimethylsilyl)-propionate-2,2,3,3-d4 [TSP] [Sigma-Aldrich], 1 mM; 80% D2O, 20% H2O). TSP served as NMR reference. Feces (200 mg) were homogenized in 800 \\u03bcl of NMR buffer for 5 min at 25 Hz in a tissue lyser (Qiagen). Plasma samples (350 \\u03bcl) were mixed with D2O (350 \\u03bcl). All homogenized samples were centrifuged (10 min, 4 degreesC, 16,000 x g) and transferred to 5-mm NMR tubes (Bruker, United Kingdom) for analysis by NMR spectroscopy. (ii) Spectrum acquisition. The NMR experiments were carried out in the Chemical Analysis Facility (CAF; University of Reading) using a Bruker AV700 NMR instrument equipped with a 5-mm inverse CryoProbe, for increased sensitivity. A standard one-dimensional nuclear Overhauser effect spectroscopy (NOESY-PR-1D) experiment was performed on all three types of samples, using a standard preset pulse sequence (noesy1d90 degrees). Additionally, a Carr-Purcell-Meiboom-Gill (CPMG) experiment was applied to plasma samples, where simple presaturation of the water peak was used. This experiment reduced the signal contribution from albumin and lipoproteins present in plasma and highlighted signals from smaller molecules. All samples were analyzed at 300 K, and a 65,000-data-point spectrum (spectral width, 14,705 Hz) was obtained by recording 128 scans (8 dummy scans). (iii) Data processing and statistical analysis. Phase and baseline of the spectra were corrected using MestreNova software (version MestreLab Research). NMR spectra were referenced to TSP peak for urine and fecal water samples and to glucose (at \\u03b4 ppm) for plasma samples. The processed spectra were digitalized and transferred to Matlab (version R2017a; MathWorks, Natick, MA) for the statistical analysis. The residual water signal was removed, and all spectra were normalized to the total spectral area for feces and urine. Plasma spectra were not normalized. Relative spectra were mean centered and scaled to unit variance. For the first stage of the analysis (unsupervised), principal-component analysis (PCA) was used. The next, supervised stage of the analysis involved the orthogonal projection to latent structure discriminant analyses (O-PLS-DA) (no orthogonal components used) to compare the changes in metabolite profiles between the two time points. The comparison was made between the baseline (start point) and the endpoint of AXOS intervention. NMR spectra were used as a matrix of variables X and time point vector Y (0, baseline; 1, endpoint) as a predictor. This analysis was used to construct a model identifying metabolites differentiating between the two time points. The internal validation of the model was evaluated using the following parameters: the goodness of fit (RY), showing what percentage of variation is explained by the model, and goodness of prediction (QY), the percentage of Y predicted after 7-cross validation. For the correlation of metabolome with microbiome data, the metagenomics readings (normalized using the \\u201creads per million\\u201d approach) were used as a predictor Y using the O-PLS analysis. The raw fastq sequences generated from the shotgun sequencing of fecal DNA are publicly available at the ENA under the project accession number .\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 13
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "metric-pleasure"
      },
      "source": [
        "Now we are updating the current project to add an extra 5 PMIDs by updating the input list."
      ],
      "id": "metric-pleasure"
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "foster-pursuit",
        "outputId": "b99777d7-c68d-42c3-821f-843206ca28b8"
      },
      "source": [
        "pmids = ['19902024', '28089709', '28601864', '31138673', '31605240', '31918705', '32586265', '33037261', '33515003', '33578731']\n",
        "\n",
        "bioscraping(pmids,\n",
        "            EMAIL, #You need to insert your email address here\n",
        "            NCBI_API_KEY, #You need to insert your NCBI_API_KEY here\n",
        "            wiley_api_key = WILEY_API_KEY, #This is an optional parameter.\n",
        "            #You can insert your WILEY_API_KEY here\n",
        "            elsevier_api_key = ELSEVIER_API_KEY, #This is an optional parameter.\n",
        "            #You can insert your ELSEVIER_API_KEY here\n",
        "            colab2 = True # This parameter is for this Notebook example only.\n",
        "            #It bypass the edirect module that can not be run on Colab.\n",
        "            )"
      ],
      "id": "foster-pursuit",
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Result for retrieved_df : \n",
            "Here is the performance thus far:\n",
            "PDF:3 = 60.0%\n",
            "XML:4 = 80.0%\n",
            "HTML:3 = 60.0%\n",
            "Plain Text:1 = 20.0%\n",
            "\n",
            "We have a tagged version (HTML, XML) for 4 articles = 80.0%\n",
            "\n",
            "We only have the abstract but not the associated content text for 0 articles = 0.0%\n",
            "\n",
            "We have a content text for 5 out of 5 articles = 100.0%\n",
            "\n",
            "\n",
            "Result for retrieved_df2 : \n",
            "Here is the performance thus far:\n",
            "PDF:7 = 70.0%\n",
            "XML:7 = 70.0%\n",
            "HTML:3 = 30.0%\n",
            "Plain Text:2 = 20.0%\n",
            "\n",
            "We have a tagged version (HTML, XML) for 7 articles = 70.0%\n",
            "\n",
            "We only have the abstract but not the associated content text for 1 articles = 10.0%\n",
            "\n",
            "We have a content text for 9 out of 10 articles = 90.0%\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "marine-mention"
      },
      "source": [
        "As you can observe from the output .i.e the difference of performance between retrieved_df and retrieved_df 2, this time, bioscraping only looked for 5 publications. The metadata_retrieved_df now counts 10 publications in total."
      ],
      "id": "marine-mention"
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "valid-scoop"
      },
      "source": [
        "with zipfile.ZipFile(\"./output/retrieved_df/retrieved_df2.json.zip\", \"r\") as z:\n",
        "    for filename in z.namelist():\n",
        "        with z.open(filename) as f:\n",
        "            data = f.read()\n",
        "            data = json.loads(data)\n",
        "\n",
        "\n",
        "f.close()\n",
        "z.close()\n",
        "metadata_retrieved_df = pd.read_json(data, orient='index')\n",
        "metadata_retrieved_df.pmid = metadata_retrieved_df.pmid.astype(str)"
      ],
      "id": "valid-scoop",
      "execution_count": 15,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "sudden-glance",
        "outputId": "a7a0eb3c-48e6-4c98-b98e-64ba4cffe56a"
      },
      "source": [
        "metadata_retrieved_df.shape"
      ],
      "id": "sudden-glance",
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(10, 25)"
            ]
          },
          "metadata": {},
          "execution_count": 16
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 620
        },
        "id": "exclusive-flour",
        "outputId": "47e5c81e-6bcb-47e4-eeb8-ab70a18a470c"
      },
      "source": [
        "metadata_retrieved_df.sample(5)"
      ],
      "id": "exclusive-flour",
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                      pmid       pmcid  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  33037261  PMC7547065   \n",
              "0b80f7eaaba9488dba9af951a818e062  31138673  PMC6538848   \n",
              "6fd66ba037574d1db4530ff3e213778b  33578731  PMC7916506   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  28601864  PMC5644969   \n",
              "8677a2c0e6214547acca432c72af98a2  28089709  PMC5357736   \n",
              "\n",
              "                                                                              title  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  Altered metabolomic profiling of overweight an...   \n",
              "0b80f7eaaba9488dba9af951a818e062  A Multi-omics Approach to Unraveling the Micro...   \n",
              "6fd66ba037574d1db4530ff3e213778b  Gut Microbiota Profile and Changes in Body Wei...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  Effect of a High-Protein Diet versus Standard-...   \n",
              "8677a2c0e6214547acca432c72af98a2  Objective assessment of dietary patterns by us...   \n",
              "\n",
              "                                                                           abstract  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  Exercise training and a healthy diet are the m...   \n",
              "0b80f7eaaba9488dba9af951a818e062  Long-term consumption of dietary fiber is gene...   \n",
              "6fd66ba037574d1db4530ff3e213778b  Gut microbiota is essential for the developmen...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  BACKGROUND: Some studies have shown that prote...   \n",
              "8677a2c0e6214547acca432c72af98a2  BACKGROUND: Accurate monitoring of changes in ...   \n",
              "\n",
              "                                                                               mesh  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  [Adolescent, Body Composition, Body Mass Index...   \n",
              "0b80f7eaaba9488dba9af951a818e062                                               None   \n",
              "6fd66ba037574d1db4530ff3e213778b                                               None   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  [Adult, Biomarkers/*blood, Blood Glucose/analy...   \n",
              "8677a2c0e6214547acca432c72af98a2  [Adult, Biomarkers/*urine, Cross-Over Studies,...   \n",
              "\n",
              "                                                                           keywords  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66                                               None   \n",
              "0b80f7eaaba9488dba9af951a818e062  [AXOS, dietary fiber, glucose homeostasis, lip...   \n",
              "6fd66ba037574d1db4530ff3e213778b  [16S sequencing, BMI, clinical trial, gut micr...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  [Diet, Metabolic syndrome, Protein intake, Wei...   \n",
              "8677a2c0e6214547acca432c72af98a2                                               None   \n",
              "\n",
              "                                                                            authors  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  [Duft RG, Castro A, Bonfante ILP, Lopes WA, da...   \n",
              "0b80f7eaaba9488dba9af951a818e062  [Benitez-Paez A, Kjolbaek L, Gomez Del Pulgar ...   \n",
              "6fd66ba037574d1db4530ff3e213778b  [Atzeni A, Galie S, Muralidharan J, Babio N, T...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64         [Campos-Nonato I, Hernandez L, Barquera S]   \n",
              "8677a2c0e6214547acca432c72af98a2  [Garcia-Perez I, Posma JM, Gibson R, Chambers ...   \n",
              "\n",
              "                                                               journal  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66                    Scientific reports   \n",
              "0b80f7eaaba9488dba9af951a818e062                              mSystems   \n",
              "6fd66ba037574d1db4530ff3e213778b                        Microorganisms   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                         Obesity facts   \n",
              "8677a2c0e6214547acca432c72af98a2  The lancet. Diabetes & endocrinology   \n",
              "\n",
              "                                                                           pub_type  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  [Journal Article, Research Support, Non-U.S. G...   \n",
              "0b80f7eaaba9488dba9af951a818e062                                  [Journal Article]   \n",
              "6fd66ba037574d1db4530ff3e213778b                                  [Journal Article]   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64     [Journal Article, Randomized Controlled Trial]   \n",
              "8677a2c0e6214547acca432c72af98a2  [Journal Article, Randomized Controlled Trial,...   \n",
              "\n",
              "                                    pub_date  ... pdf xml  html plain pmc_tgz  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  2020-10-09  ...   1   1     1     0       1   \n",
              "0b80f7eaaba9488dba9af951a818e062  2019-05-28  ...   1   1     0     0       1   \n",
              "6fd66ba037574d1db4530ff3e213778b  2021-02-10  ...   1   1     0     0       1   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64  2017-11-24  ...   0   0     0     0       0   \n",
              "8677a2c0e6214547acca432c72af98a2  2017-03-24  ...   1   0     0     1       1   \n",
              "\n",
              "                                                                        xml_parse_d  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  {'file_path': './output/formats/xmls/d5bfee539...   \n",
              "0b80f7eaaba9488dba9af951a818e062  {'file_path': './output/formats/xmls/0b80f7eaa...   \n",
              "6fd66ba037574d1db4530ff3e213778b  {'file_path': './output/formats/xmls/6fd66ba03...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "8677a2c0e6214547acca432c72af98a2                                                 {}   \n",
              "\n",
              "                                                                       html_parse_d  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  {'file_path': './output/formats/htmls/d5bfee53...   \n",
              "0b80f7eaaba9488dba9af951a818e062                                                 {}   \n",
              "6fd66ba037574d1db4530ff3e213778b                                                 {}   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "8677a2c0e6214547acca432c72af98a2                                                 {}   \n",
              "\n",
              "                                                                        pdf_parse_d  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66  {'file_path': './output/formats/pdfs/d5bfee539...   \n",
              "0b80f7eaaba9488dba9af951a818e062  {'file_path': './output/formats/pdfs/0b80f7eaa...   \n",
              "6fd66ba037574d1db4530ff3e213778b  {'file_path': './output/formats/pdfs/6fd66ba03...   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "8677a2c0e6214547acca432c72af98a2  {'file_path': './output/formats/pdfs/8677a2c0e...   \n",
              "\n",
              "                                                                      plain_parse_d  \\\n",
              "d5bfee539f74496da9d6f0fa639c8d66                                                 {}   \n",
              "0b80f7eaaba9488dba9af951a818e062                                                 {}   \n",
              "6fd66ba037574d1db4530ff3e213778b                                                 {}   \n",
              "e937a9417a6e4723bdb6d07e4cd71a64                                                 {}   \n",
              "8677a2c0e6214547acca432c72af98a2  {'file_path': './output/formats/txts/8677a2c0e...   \n",
              "\n",
              "                                  content_text  \n",
              "d5bfee539f74496da9d6f0fa639c8d66             1  \n",
              "0b80f7eaaba9488dba9af951a818e062             1  \n",
              "6fd66ba037574d1db4530ff3e213778b             1  \n",
              "e937a9417a6e4723bdb6d07e4cd71a64             0  \n",
              "8677a2c0e6214547acca432c72af98a2             1  \n",
              "\n",
              "[5 rows x 25 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-f0d1ef25-b75a-4e3b-94c8-a1a38bd722e3\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>pmcid</th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>mesh</th>\n",
              "      <th>keywords</th>\n",
              "      <th>authors</th>\n",
              "      <th>journal</th>\n",
              "      <th>pub_type</th>\n",
              "      <th>pub_date</th>\n",
              "      <th>...</th>\n",
              "      <th>pdf</th>\n",
              "      <th>xml</th>\n",
              "      <th>html</th>\n",
              "      <th>plain</th>\n",
              "      <th>pmc_tgz</th>\n",
              "      <th>xml_parse_d</th>\n",
              "      <th>html_parse_d</th>\n",
              "      <th>pdf_parse_d</th>\n",
              "      <th>plain_parse_d</th>\n",
              "      <th>content_text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>d5bfee539f74496da9d6f0fa639c8d66</th>\n",
              "      <td>33037261</td>\n",
              "      <td>PMC7547065</td>\n",
              "      <td>Altered metabolomic profiling of overweight an...</td>\n",
              "      <td>Exercise training and a healthy diet are the m...</td>\n",
              "      <td>[Adolescent, Body Composition, Body Mass Index...</td>\n",
              "      <td>None</td>\n",
              "      <td>[Duft RG, Castro A, Bonfante ILP, Lopes WA, da...</td>\n",
              "      <td>Scientific reports</td>\n",
              "      <td>[Journal Article, Research Support, Non-U.S. G...</td>\n",
              "      <td>2020-10-09</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>{'file_path': './output/formats/xmls/d5bfee539...</td>\n",
              "      <td>{'file_path': './output/formats/htmls/d5bfee53...</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/d5bfee539...</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>0b80f7eaaba9488dba9af951a818e062</th>\n",
              "      <td>31138673</td>\n",
              "      <td>PMC6538848</td>\n",
              "      <td>A Multi-omics Approach to Unraveling the Micro...</td>\n",
              "      <td>Long-term consumption of dietary fiber is gene...</td>\n",
              "      <td>None</td>\n",
              "      <td>[AXOS, dietary fiber, glucose homeostasis, lip...</td>\n",
              "      <td>[Benitez-Paez A, Kjolbaek L, Gomez Del Pulgar ...</td>\n",
              "      <td>mSystems</td>\n",
              "      <td>[Journal Article]</td>\n",
              "      <td>2019-05-28</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>{'file_path': './output/formats/xmls/0b80f7eaa...</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/0b80f7eaa...</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6fd66ba037574d1db4530ff3e213778b</th>\n",
              "      <td>33578731</td>\n",
              "      <td>PMC7916506</td>\n",
              "      <td>Gut Microbiota Profile and Changes in Body Wei...</td>\n",
              "      <td>Gut microbiota is essential for the developmen...</td>\n",
              "      <td>None</td>\n",
              "      <td>[16S sequencing, BMI, clinical trial, gut micr...</td>\n",
              "      <td>[Atzeni A, Galie S, Muralidharan J, Babio N, T...</td>\n",
              "      <td>Microorganisms</td>\n",
              "      <td>[Journal Article]</td>\n",
              "      <td>2021-02-10</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>{'file_path': './output/formats/xmls/6fd66ba03...</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/6fd66ba03...</td>\n",
              "      <td>{}</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>e937a9417a6e4723bdb6d07e4cd71a64</th>\n",
              "      <td>28601864</td>\n",
              "      <td>PMC5644969</td>\n",
              "      <td>Effect of a High-Protein Diet versus Standard-...</td>\n",
              "      <td>BACKGROUND: Some studies have shown that prote...</td>\n",
              "      <td>[Adult, Biomarkers/*blood, Blood Glucose/analy...</td>\n",
              "      <td>[Diet, Metabolic syndrome, Protein intake, Wei...</td>\n",
              "      <td>[Campos-Nonato I, Hernandez L, Barquera S]</td>\n",
              "      <td>Obesity facts</td>\n",
              "      <td>[Journal Article, Randomized Controlled Trial]</td>\n",
              "      <td>2017-11-24</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8677a2c0e6214547acca432c72af98a2</th>\n",
              "      <td>28089709</td>\n",
              "      <td>PMC5357736</td>\n",
              "      <td>Objective assessment of dietary patterns by us...</td>\n",
              "      <td>BACKGROUND: Accurate monitoring of changes in ...</td>\n",
              "      <td>[Adult, Biomarkers/*urine, Cross-Over Studies,...</td>\n",
              "      <td>None</td>\n",
              "      <td>[Garcia-Perez I, Posma JM, Gibson R, Chambers ...</td>\n",
              "      <td>The lancet. Diabetes &amp; endocrinology</td>\n",
              "      <td>[Journal Article, Randomized Controlled Trial,...</td>\n",
              "      <td>2017-03-24</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>{}</td>\n",
              "      <td>{}</td>\n",
              "      <td>{'file_path': './output/formats/pdfs/8677a2c0e...</td>\n",
              "      <td>{'file_path': './output/formats/txts/8677a2c0e...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>5 rows √ó 25 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-f0d1ef25-b75a-4e3b-94c8-a1a38bd722e3')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-f0d1ef25-b75a-4e3b-94c8-a1a38bd722e3 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-f0d1ef25-b75a-4e3b-94c8-a1a38bd722e3');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-06dd36e8-1cfc-49f3-9b11-3826ca3ef259\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-06dd36e8-1cfc-49f3-9b11-3826ca3ef259')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-06dd36e8-1cfc-49f3-9b11-3826ca3ef259 button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe"
            }
          },
          "metadata": {},
          "execution_count": 17
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df = parsed_to_df()"
      ],
      "metadata": {
        "id": "vXPgSOek5vBk"
      },
      "id": "vXPgSOek5vBk",
      "execution_count": 18,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Only the lines where the full-text was retrieved are included."
      ],
      "metadata": {
        "id": "pnBIHyCWviS8"
      },
      "id": "pnBIHyCWviS8"
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "k8AATIY_5vEI",
        "outputId": "3ad3a84e-7a02-4712-83ca-8f7227459fc4"
      },
      "id": "k8AATIY_5vEI",
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(9, 1)"
            ]
          },
          "metadata": {},
          "execution_count": 19
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "retrieved_df.sample(5)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "BiH6ZASh5vGu",
        "outputId": "2a9effb8-ad22-4036-8684-43a1637e90bf"
      },
      "id": "BiH6ZASh5vGu",
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                       content_text\n",
              "0b80f7eaaba9488dba9af951a818e062  Long-term consumption of dietary fiber is gene...\n",
              "84927ce9fe4d49a881716ec697a76af7  BACKGROUND: Epidemiologic studies show that co...\n",
              "8ff3e24cb260420498b5537e372a1a1e  INTRODUCTION: Low gut microbiome richness is a...\n",
              "7575147222f842aba0727e3632c8f6dc  Low caloric diet (LCD) is used for weight loss...\n",
              "4fa376bec991422899c69fdfb7616146    S0002-9165(22)00623-2 Maternal gut microbiot..."
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-ae912bb3-0308-4b99-a0c7-1e85cdff8d9a\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>content_text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0b80f7eaaba9488dba9af951a818e062</th>\n",
              "      <td>Long-term consumption of dietary fiber is gene...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>84927ce9fe4d49a881716ec697a76af7</th>\n",
              "      <td>BACKGROUND: Epidemiologic studies show that co...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8ff3e24cb260420498b5537e372a1a1e</th>\n",
              "      <td>INTRODUCTION: Low gut microbiome richness is a...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7575147222f842aba0727e3632c8f6dc</th>\n",
              "      <td>Low caloric diet (LCD) is used for weight loss...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4fa376bec991422899c69fdfb7616146</th>\n",
              "      <td>S0002-9165(22)00623-2 Maternal gut microbiot...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ae912bb3-0308-4b99-a0c7-1e85cdff8d9a')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-ae912bb3-0308-4b99-a0c7-1e85cdff8d9a button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-ae912bb3-0308-4b99-a0c7-1e85cdff8d9a');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-11c86d00-e4f1-484c-8747-bc849f010554\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-11c86d00-e4f1-484c-8747-bc849f010554')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-11c86d00-e4f1-484c-8747-bc849f010554 button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "summary": "{\n  \"name\": \"retrieved_df\",\n  \"rows\": 5,\n  \"fields\": [\n    {\n      \"column\": \"content_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 5,\n        \"samples\": [\n          \"BACKGROUND: Epidemiologic studies show that cocaine- and amphetamine-regulated transcript prepropeptide (CARTPT) gene polymorphism modifies diet-obesity relationships. However, the interaction between CARTPT gene polymorphism and diet quality indices have not been investigated yet. The current study was aimed to evaluate the interaction between major dietary indices including Diet Quality Index-International (DQI-I) and Healthy Eating Index (HEI)-2015 and CARTPT gene rs2239670 variants among apparently healthy obese Iranians. METHODS: This cross-sectional study was carried out by employing 288 apparently healthy obese adults aged 20-50 years with a BMI of 30-40 kg/m(2). Diet quality was evaluated by Diet Quality Index-International (DQI-I) and Healthy Eating Index-2015 (HEI-2015) using a 132-items semi-quantitative validated food frequency questionnaire. The CARTPT gene rs2239670 polymorphism was genotyped by polymerase chain reaction-restriction fragment length polymorphism (PCR-RFLP) technique. Blood concentrations of glycemic markers, lipid profile, alpha-melanocyte stimulating hormone (MSH) and agouti-related peptide (AgRP) were also measured. ANCOVA multivariate interaction model was used to analyze gene-diet interactions. RESULTS: The significant interactions were identified between CARTPT gene polymorphism and HEI, affecting BMR (P(Interaction) = ), serum glucose (P(Interaction) = ) and high density lipoprotein cholesterol HDL concentrations (P(Interaction) = ) after adjusting for the effects of sex and age. Also we found gene-diet interaction between CARTPT genotypes and DQI-I in terms of fat mass (FM; P(Interaction) = ), waist circumference (WC; P(Interaction) < ), body mass index (BMI; P(Interaction) < ), basal metabolic rate (BMR, P(Interaction) < ), serum fasting glucose (P(Interaction) < ) and AgRP (P(Interaction) = ) in individuals even after adjusting for potential confounders. CONCLUSION: Current study showed the effects of interaction between CARTPT genotype with adherence to HEI and DQI-I scores on obesity-related anthropometric and metabolic risk-factors. Obesity, as a major pandemic health problem, refers to the excessive and abnormal fat accumulation according to the definition of World Health Organization (WHO) [], resulting from an imbalance between energy intake and expenditure []. It is mostly defined as having body mass index (BMI) greater than 30 kg/m []. The prevalence of obesity has been increased dramatically in the past decades []. It has been estimated that approximately billion and 573 million individuals will be obese and overweight by 2030 []. Recent studies show that obesity prevalence have been raised in Iranian adults in last years []. The interaction between genetic and life style behaviors including diet and physical activity have been recently attracted much attention in treatment and management of obesity and related disorders. Genetic and molecular epidemiological evidences suggest that there is a clear difference in the susceptibility to obesity and obesity-related disorders according to their genetic traits []. Accordingly, the adverse effects of obesity on cardio-metabolic risk factors has strong dependence to genetic variants and therefore, it is necessary to develop a tri-polar approach of gene-nutrient-metabolic risk factor interactions in molecular studies. So, nutrigenetics and nutrigenomics studies can provide an understanding of the intra and inter population phenotypic differences in the effect of obesity on metabolic risk factors []. Over the past decades, genome-wide association studies (GWAS) have identified the cocaine- and amphetamine-regulated transcript prepropeptide gene encoding cocaine- and amphetamine-regulated transcript (CART) protein as a neurotransmitter and hormone [\\u2013], located in the arcuate nucleus of the hypothalamus [] and implicated in several physiological processes such as neurological disease, anxiety, stress, addiction, depression, endocrine function, specially feeding behavior, weight management and energy homeostasis [\\u2013]. CART prepropeptide gene located in chromosome 5q13-14 in coexistence of proopiomelanocortin (POMC) neurons [], contains one exon and two introns [] and produces short peptide fragments [, ] (Fig. ). Alpha melanocyte-stimulating hormone (\\u03b1-MSH) as a derivative of POMC along with CART leads to energy expenditure enhancing food intake and appetite inhibition []. It has been identified that CART prepropeptide gene polymorphism is associated with obesity []. Also Rigoli  [] declared that CART gene single nucleotide polymorphism (SNP) may affect CART expression which is associated with obesity in children. Other study by del-Giudice  [] suggested the possible role of Leu34Phe mutation of CART gene might play an important role in obese phenotype by altering the CART mediated leptin effect on thermogenesis and energy expenditure. Likewise, understanding of gene-diet interactions may be more effective in prevention of the obesity-related phenotypes and their better management. The role of diet as one of lifestyle factors in preventing obesity and relevant non-communicable diseases and also maintaining good health is indisputable []. Since food and food ingredients including micro or macro-nutrients are usually consumed in combination of each other in a usual diet and as a consequent they will have numerous synergistic or even inhibitory interactions, it is logical to study their simultaneous effects in the diet rather than studying their isolate actions; dietary pattern approach, accounting complexity of the diet, provides the opportunity to approve the real image of a diet on health outcome []. Among priori-defined dietary pattern approaches, as known as \\u2018diet quality indices\\u2019, including diet quality index International (DQI-I) and healthy eating index (HEI) [, ] have been known as being cardio-protective. DQI-I is developed to clarify dietary variations among population; whereas, HEI-2015, the most recently updated version is a nutrient density-based approach []. In the current study, we aimed to evaluate the interaction between CARTPT genotype and dietary indices and the effects of these interactions on anthropometric and metabolic risk factors in obese individuals. The current cross sectional study was conducted in Tabriz, Iran between December 2017 and April 2019. Using convenience random sampling procedure, 288 healthy obese individuals who met the inclusion criterias recruited to the study. Eligible individuals aged 20\\u201350 years old with a BMI of 30\\u201340 kg/m without prior history of alcohol or drug abuse, any presence of acute or chronic inflammatory disease, hypertension, hepatic, renal or cardiovascular disorders, diabetes mellitus, malignancies, thyroid diseases, and acute or chronic infections. Subjects with a history of weight change more than 5 kg in the last 6 months, use of any medications affecting weight, pregnancy and lactation were excluded from the study. Written informed consent was obtained from all the participants at the beginning of the study. The study protocol was approved and registered by the ethics committee of Tabriz University of Medical Sciences (Ethics number: ). Demographic data including: age, sex, socioeconomic status (SES), marital status, smoking and medical history were completed for each individual in a structured questionnaire by interviewing. Anthropometric assessments including weight, height, waist circumference (WC), and hip circumference (HC) were performed by a trained person. Weight was measured with a Seca scale to the nearest 100 g removing his/her shoes with a lightweight clothing. Height was measured to the nearest cm using a local stadiometer in a normal standing position. WC was measured at the narrowest area of waist using a non-stretchable tape measure with a cm precision and without any pressure to the body. HC was also measured around the most prominent part of the hip to the nearest cm. Waist-to-hip ratio (WHR) was calculated by dividing WC to HC. Systolic and diastolic blood pressure (SBP and DBP) were assessed twice in a relaxing position after 15 min-resting and the mean values were recorded. Body composition was measured through body composition analyzer BC-418-Tanita (United Kingdom). Physical activity level was also evaluated using the international physical activity questionnaire (IPAQ). Intravenous blood was collected from all the subjects after 12 h fasting and centrifuged 10 min at 3000 rpm, 4 degreesC, aliquoted into 1 mL tubes and stored at \\u2212 80 degreesC until assay. Fasting serum glucose, triglyceride (TG), total cholesterol and high-density lipoprotein cholesterol (HDL-C) were measured using a commercial kit (Pars Azmoon, Tehran, Iran). Serum low-density lipoprotein cholesterol (LDL-C) was calculated according to the Friedewald method []. Plasma \\u03b1-MSH and AgRP levels were measured using commercially available enzyme-linked immunosorbent assay (ELISA) kits (Bioassay Technology Laboratory, Shanghai Korean Biotech, Shanghai City, China) according to the manufacturer\\u2019s instructions. The minimum detectable doses of \\u03b1-MSH and AgRP levels were ng/L and pg/mL, respectively. Serum insulin level was determined by similar ELIZA kit (Bioassay Technology Laboratory, Shanghai Korean Biotech, Shanghai City, China). Homeostasis model assessment-insulin resistance index (HOMA-IR) and quantitative insulin sensitivity check index (QUICKI) were computed according to the following equations []:HOMA-UR=fasting glucosemmol/Lxfasting insulin\\u03bcIU/mL/ glucosemmol/L+logfasting insulin\\u03bcIU/mL Dietary intake was evaluated by using 132-items semi-quantitative food frequency questionnaire (FFQ) a reliable and validated questionnaire to estimate the food consumption and dietary nutrient intake over the past year []. Questionnaires were completed by a well-trained nutritionist, and participants reported the intake frequency and quantity of each food item over the past year. Then, portion sizes of reported foods were converted to gram due to household measurements. Ultimately daily energy and nutrient intake were analyzed using Iranian food composition table (FCT) []. The Healthy Eating Index (HEI) is a method for diet quality assessment according to dietary guidelines for Americans (DGA) recommendations []. It has been structured to examine diet quality and health outcomes associations. HEI-2015 is composed of 13 components, 9 adequacy and 4 moderation components with a total score of 100 points. The highest and lowest consumption of three adequacy components (., whole grains, dairy and fatty acids) scored 10 and 0, respectively. Other Six adequacy components include total fruits (fruit, fruit juice and canned fruit), whole fruits (fruits except fruit juice), total vegetables, seafood and plant proteins, greens and beans, and total protein foods; each scored at 5 for the highest and 0 for the lowest consumption. A maximum of 10 points was given to the lowest consumption of four moderation components includes refined grains, sodium, added sugars, and saturated fats. However, the highest consumption of these items was scored as 0. Intermediate intake of every component was scored proportionally. Higher overall HEI-2015 scores shows greater alignment with DGA recommendation and better diet quality []. The Diet Quality Index\\u2014International (DQI-I) is a measure determining overall health effects of diet and provides appropriate nutrition intake recommendations. It takes into account both undernutrition and overnutrition statues. DQI-I could evaluate diet quality in terms of nutritional variety, adequacy, moderation and balance together. Dietary variety scores such as diversity in food groups and protein sources gets 20 points. Adequacy scores contain adequate fruits, vegetables, grains intake obtain 40 points. Moderation scores includes total fat, saturated fat, cholesterol, sodium wile empty calorie foods gets 30 points and lastly overall balance score which consider macronutrient balance and fatty acid ratios receives 10 points. Scores of each components in four categories will be summed to obtain the final DQI score. A higher DQI-I score is considered as a high quality diet indicator []. Blood samples were collected from all individuals in EDTA-containing tubes. Genomic DNA was extracted from blood sample and its concentration was measured using Nano Drop 2000C spectrophotometer. The CARTPT rs2239670 SNP (major allele: G; minor allele: A) was genotyped by polymerase chain reaction-restriction fragment length polymorphism (PCR\\u2013RFLP) technique. The forward and backward primers were 5\\u2032-CCTGCTGCTGATGCTACCTCT-3\\u2032 and 5\\u2032-GCGCTTCGATCTGCAACACAC-3\\u2032, respectively. PCR was run as follows: denaturation at 94 degreesC for 30 s, annealing at 60 degreesC for 30 s, and extension at 72 degreesC for 20 s, for 35 cycles. The reaction cycles were preceded by a single cycle of denaturation at 94 degreesC for 5 min and ended with a single cycle of extension at 72 degreesC for 10 min. The RFLP analysis for the rs2239670 variant was performed by incubating the PCR products with ApaI (Takara, Japan) enzyme. All products were electrophoresed on 3% agarose gels. Fragments containing three possible genotypes were then distinguished: uncut homozygous AA (552 bp), cut heterozygous AG (212, 340 and 552 bp) and cut homozygous GG (340 and 212 bp). Data were analyzed using SPSS version . Descriptive measures such as coefficients of skewness and kurtosis, mean and standard deviation (SD) were used to check distribution of data. We used the mean +/- SD for reporting normally distributed continuous variables, the median (25th and 75th percentile) for the continuous variables that were not normally distributed, and the frequency (%) for categorical data. General characteristics of subjects across CARTPT genotypes were determined using one-way analysis of variance (ANOVA) and Chi square tests. Multivariate multinomial logistic regression was performed to check out diet quality indices and rs14477313 genotype associations. ANCOVA multivariate model with adjustment to confounders using the general linear model procedure have been used to investigate CART polymorphism (rs2239670) and diet quality indices interactions. P values less than were considered statistically significant. A total of 288 participants were involved in this cross sectional study with sex distribution of and in men and women respectively and the allele frequency of A (), G (). Basic characteristics of study participants across the CART rs2239670 genotypes have been presented in Table . Genotype prevalence among study participants was as follow: AA (), AG (), GG () and AA (), AG (20%), GG () in men and women respectively. Significant differences had been shown about age distribution among genotypes in men (P < ). Table indicates biochemical parameters of participants according to CARTPT gene rs2239670 polymorphisms. There were statistically significant difference in terms of \\u03b1-MSH (P = ), AgRP (P = ) in women and glucose (P = ) and HOMA (P = ) in men across CARTPT gene rs2239670 genotypes. While higher amounts of these parameters were observed in AA genotype compared with other genotypes. Figure presents CARTPT-HEI interactions and their effects on several anthropometric and biochemical variables. Accordingly, CARTPT-HEI interactions affect basal metabolic rate (BMR; PInteraction = ), glucose (PInteraction = ), HDL (PInteraction = ) after adjusting for sex and age. The risk of obesity and metabolic syndrome was not homogenous in CART genotype groups, across tertiles of HEI. The lowest BMR was observed in the AG genotype with moderate compliance with the HEI; highest BMR values was observed in highest HEI tertile in AA and AG genotypes. AA genotype had also the highest fasting serum glucose concentrations even in highest adherence to HEI. While being in AG or GG genotype reduced serum glucose concentrations (P = ). Also the lowest HDL was observed in the AG genotype with moderate adherence to the HEI; while its concentrations reduced in highest tertile. As a conclusion, HEI could not modify the adverse effects of being in AA genotype of CARTPT. Whereas, gene-diet interactions were more pronounced for CARTPT DQI-I compared with CARTPT-HEI as represented for FM (PInteraction = ), WC (PInteraction < ), BMI (PInteraction < ), BMR (PInteraction < ), serum fasting glucose (P Interaction < ) and AgRP (PInteraction = ) even after adjusting for potential confounders (Fig. ). Higher adherence to DQI-I scores reduced fasting serum glucose and Ag-RP levels in AA genotype; while individuals with lower adherence to DQI-I exhibited a higher serum fasting glucose and Ag-RP concentrations. Accordingly, highest adherence to DQI-I index reduced FM, BMI, fasting serum glucose and Ag-RP in AA genotype while increased BMR values further highlighting the beneficial effects of high adherence to DQI-I to reduce the adverse obesity-related risk factors. The interaction between HEI or DQI-I and other anthropomeric or biochemical parameters were not statistically significant while the related information are provided in supporting information (Additional file : Figures S1 and S2). To the best of our knowledge, this is the first study investigated the interactions between CARTPT rs2239670 genes with dietary indices in terms of metabolic parameters. We demonstrated that AA genotype was the possible risk factor of metabolic disorders and adherence to higher diet quality index of DQI-I reduced the metabolic risk in obese individuals. While these interactive role was not favorite for HEI. As elucidated above AA homozygote genotype can increase metabolic abnormalities like serum fasting glucose level despite high adherence to HEI and DQI-I and also can increase FM, BMI, AgRP even in high compliance with DQI-I. The role of CARTPT gene in obesity has been studied in several previous works; Two variants, 1457delA and A1475G, in the coding region of CARTPT gene were detected in Danish Caucasians with early onset obesity in Echwald  study []. In addition, according to the results of Challis study, the A1475G and 1457delA variants are not associated with obesity in Caucasian population in the , and the A1475G SNP was significantly associated with lower WHR, fasting plasma insulin, and fasting triglycerides []. Likewise, no association was reported between C1442G variant of CARTPT gene and obesity in a group of obese Pima Indians []. In contrast, Yamada  [] demonstrated that the A-156G variant of the CARTPT gene was associated with obesity in the Japanese population. Accordingly, our results were in line with other Asian populations in Korea [] and Malaysia [] where the majority of the participants had GG genotype about the genotype prevalence and allele frequency of this gene polymorphism. Moreover, consistent with our findings, there was no significant difference in term of anthropometric measurements and blood pressures among the genotypes []. Although the underlying mechanisms for these different results remain poorly understood, differences in dietary habits and other socio-demographic factors in studied populations may contribute to these inconsistent outcomes. Although, no study is available evaluating the CARTPT gene-interactions with dietary indices and their effects on metabolic parameters in obesity, but numerous previous studies are available investigating the interaction between single nucleotide gene polymorphisms with diet or dietary ingredients in the pathogenesis of obesity-related comorbidities; for example the interaction between Mediterranean dietary pattern and FTO gene variants [], dietary inflammatory index and VEGF genotype [], and the role of dietary approach to stop hypertension interaction with insulin receptor substrate-1gene expression [] affecting obesity and obesity related metabolic phenotypes are available in literature. Our findings revealed the interactive role of DQI-I in reducing obesity related metabolic profile including BMI, FM and FSG and AgRP in AA genotype of CARTPT; no human study is available evaluating the relationship between AgRP and CART gene polymorphisms. However, increased AgRP in obesity had been reported in several previous reports [\\u2013]. The possible underlying mechanism of increased AgRP in obesity is hypothalamic re-sistance to its function due to increased leptin concentrations in obese individuals []. In terms of CARTPT-HEI interactions, being in highest tertiels of HEI could not show favorable effects in reducing metabolic risk factors; for example FM increased while HDL decreased in highest adherence to HEI in AA genotype. Similarly, Sanjeevi  and Kirwan  reported that a whole grain-rich diet with high adherence to HEI score significantly reduced serum HDL concentrations in overweight and obese adults [, ]. To date, there is no study regarding the interaction between the CARTPT polymorphisms and dietary indices in relation to cardio-metabolic traits and hypothalamic hormones in obesity and this is the main strength of current study. Our report, partially, emphasize to the beneficial role of higher adherence to DQI-I in reducing the obesity-related risk factors. There are several limitations in this study that should be taken into account. First, the cross-sectional design of the study makes it impossible to infer causality; therefore, more studies with interventional designs are warranted. Second, under-reporting of dietary intakes in obese adults, could be a possible source of report and recall bias affecting the results [], although the FFQ used in the current study is a validated questionnaire and the information are reliable; moreover, FFQ covers a wide range of dietary ingredients and has more accuracy compared with 24-h recall method in reflection of usual dietary intake in a short period of time; it has been confirmed that FFQ could be more helpful in evaluating the diet-disease relationships []. Our findings revealed an association between the rs2239670 SNP of the CARTPT gene and obesity and exhibited evidences several interactions between this variant and dietary indices in terms of BMR, WC, FM, serum glucose and AgRP particularly in terms of CARTPT\\u2013DQI-I associations. \",\n          \"  S0002-9165(22)00623-2 Maternal gut microbiota reflecting poor diet quality is associated with spontaneous preterm birth in a prospective cohort study The American Journal of Clinical Nutrition Journal fla 00029165 113 3 602 611 602-611 3 text/plain 2021-03-31 March 2021 Copyright 2021 American Society for Nutrition. Published by Elsevier Inc. All rights reserved. American Society for Nutrition. Published by Elsevier Inc. Gershuni, Victoria Li, Yun Elovitz, Michal Li, Hongzhe Wu, Gary D Compher, Charlene W Background A processed diet, high in fat and low in fiber, is associated with differences in the gut microbiota and adverse health outcomes in humans; however, little is known about the dietmicrobiota relation and its impact on pregnancy. Spontaneous preterm birth (SPTB), a pregnancy outcome with serious shortand long-term consequences, occurs more frequently in black and in obese women in the United States. Objectives In a prospective, case-control sample matched for race and obesity (cases = 16, controls = 32), we compared the fecal gut microbiota, fecal and plasma metabolites, and diet in the late second trimester. We hypothesized that a Western diet would be associated with reduced microbiota richness and a metabolic signature predicting incidence of SPTB. Methods The fecal microbiota was characterized by 16S-tagged sequencing and untargeted metabolomics was used to analyze both plasma and fecal metabolites. Wilcoxons rank-sum test was used for the comparison of microbiota genera,diversity, fecal and plasma metabolites, and dietary variables between term and was analyzed using permutational multivariate ANOVA, and metabolite associations were assessed by module analysis. Results A decrease indiversity was strongly associated with the development of SPTB, especially in the taxonomic class of Betaproteobacteria. Of 824 fecal metabolites, 22 metabolites (mostly lipids) differed between cases and controls (P < ), with greater DHA (22:6n3) and EPA (20:5n3) in cases [false discovery rate (FDR) < ]. The most significant fecal metabolite module (FDR-adjusted P = ) was dominated by DHA and EPA. Dietary saturated fat (primarily palmitate) intake was greater in cases ( compared with  g, P = ) and was positively correlated with fecal DHA and EPA (P < ). Conclusions Reduceddiversity of the gut microbiota and higher excretion of omega-3 (n3) fatty acids in stool may provide a novel biomarker signature predicting SPTB in women with a low-fiber, high-fat diet. Further investigation of these markers in a larger sample is needed for validation. Published by Elsevier Inc. All rights reserved. Copyright 2021 American Society for Nutrition. FIGURE 1 Shannon diversity and richness of the gut microbiota at 2026 weeks of gestation in women who spontaneously delivered preterm (n = 16) relative to those with term deliveries (n = 32). (A) Shannon diversity (Wilcoxons rank-sum test, P = ). (B) Microbiota richness (Wilcoxons rank-sum test, P = ). (C) Menhinicks index (Wilcoxons rank-sum test, P = ). In each plot the bar is the median value, and the whiskers indicate the 25% and 75% distribution. SPTB, spontaneous preterm birth. FIGURE 1 FIGURE 2 Shannon diversity of bacterial classes at 2026 weeks of gestation in women who spontaneously delivered preterm (n = 16) relative to those with term deliveries (n = 32). (A) The composition of mean Shannon diversity by bacterial class. (B) Shannon diversity of Betaproteobacteria in SPTB compared with term (Wilcoxons rank-sum test, false discovery rate < ). In each plot the bar is the median value, and the whiskers indicate the 25% and 75% distribution. (C) AUC in a receiver operating characteristic analysis using a logistic regression model to predict SPTB with the contribution of Shannon diversity of Betaproteobacteria (AUC = ) and of the overall microbiota (AUC = ). SPTB, spontaneous preterm birth. FIGURE 2 FIGURE 3 Principal components analysis plot ofdiversity between women who later experienced SPTB (n = 16) and term controls (n = 32). (A) The phylogenetic distance of proportional abundance (weighted Unifrac, P = ) and (B) presence-absence (unweighted Unifrac, P = ), analyzed by permutational multivariate ANOVA in SPTB and term. SPTB, spontaneous preterm birth. FIGURE 3 FIGURE 4 Untargeted fecal metabolites in cases and controls. (A) Of 824 fecal metabolites, 22 were differentially abundant in women with SPTB compared with term deliveries (P < level and & = FDR < ). (B) Heatmap comparing partial Kendall correlations between differentially abundant dietary micronutrients in columns and fecal metabolites in rows. ,Significant correlation: P < P < . (C) The most significantly associated module, built around3 fatty acid metabolites, identified by Weighted Gene Co-expression Network Analysis of fecal metabolites (FDR = ). Individual metabolites are represented by nodes, and are connected by edges whose topological overlap is above the threshold of . The centrality of a node (the number of adjacent edges) is represented by the node size. FDR, false discovery rate; SPTB, spontaneous preterm birth. Supported by the University of Pennsylvania March of Dimes Prematurity Research Center (to ME). The project was also supported in part by National Center for Research Resources and National Center for Advancing Translational Sciences, NIH grant UL1TR000003, the PennCHOP Microbiome Program, and Center for Molecular Studies in Digestive and Liver Disease grant NIH P30 DK050306. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. Neither funding source influenced the study design or findings. Supplemental Figures 17 and Supplemental Tables 1 and 2 are available from the Supplementary data link in the online posting of the article and from the same link in the online table of contents at . VG and YL contributed equally to this work as first authors. GDW and CC contributed equally to this work as senior authors. Spontaneous preterm birth (SPTB), a pregnancy outcome with serious shortand long-term consequences, occurs more frequently in black and in obese women in the United States. Objectives In a prospective, case-control sample matched for race and obesity (cases = 16, controls = 32), we compared the fecal gut microbiota, fecal and plasma metabolites, and diet in the late second trimester. We hypothesized that a Western diet would be associated with reduced microbiota richness and a metabolic signature predicting incidence of SPTB. Methods The fecal microbiota was characterized by 16S-tagged sequencing and untargeted metabolomics was used to analyze both plasma and fecal metabolites. Wilcoxons rank-sum test was used for the comparison of microbiota genera,diversity, fecal and plasma metabolites, and dietary variables between term and was analyzed using permutational multivariate ANOVA, and metabolite associations were assessed by module analysis. Results A decrease indiversity was strongly associated with the development of SPTB, especially in the taxonomic class of Betaproteobacteria. Of 824 fecal metabolites, 22 metabolites (mostly lipids) differed between cases and controls (P < ), with greater DHA (22:6n3) and EPA (20:5n3) in cases [false discovery rate (FDR) < ]. The most significant fecal metabolite module (FDR-adjusted P = ) was dominated by DHA and EPA. Dietary saturated fat (primarily palmitate) intake was greater in cases ( compared with  g, P = ) and was positively correlated with fecal DHA and EPA (P < ). Conclusions Reduceddiversity of the gut microbiota and higher excretion of omega-3 (n3) fatty acids in stool may provide a novel biomarker signature predicting SPTB in women with a low-fiber, high-fat diet. Further investigation of these markers in a larger sample is needed for validation. Keywords: microbiome metabolome pregnancy Western diet precision nutrition Abbreviations used: DHEA-S dehydroepiandrosterone sulfate DPA docosapentaenoic acid FDR false discovery rate HDoHE hydroxydocosahexanoic acid PERMANOVA permutational multivariate analysis of variance PTB preterm birth SPTB spontaneous preterm birth TOM topological overlap measure WGCNA Weighted Gene Co-expression Network Analysis. Introduction Preterm birth (PTB) occurs before the 37th week of gestation and is one of the leading causes of neonatal morbidity and mortality. Evidence suggests that PTB is a multifactorial syndrome potentially mediated by infection, oxidative stress, inflammation, and maternal perinatal nutrition (1 , 2 ); however, the relation is unclear and the biology of PTB remains poorly understood. Whereas the incidence of infants delivered before 37 weeks of gestation had been on the decline from 2004 to 2013, the percentage of PTB deliveries has recently increased dramatically to 10% as of 2018, with especially high percentages among black women (14%) (3 ). Obese women are also at high risk of PTB (4 ). This is particularly concerning because PTB is responsible for nearly one-third of neonatal deaths in the United States (5 ), and infants born preterm may have serious long-term health conditions due to their immaturity (6 ), including respiratory distress syndrome, necrotizing enterocolitis, metabolic disease, and the potential for lifelong complications. Shockingly, rates of PTB in the United States are considerably greater than in other similarly developed societies, such as the Nordic countries which have a PTB rate of only 6% (7 ). Within this context, the identification of biomarkers that predict PTB would be of considerable utility, especially if they provide new insights into pathogenesis, which could lead to novel approaches to reduce risk. There has been increasing interest in the gut microbiota and its association with health and disease in human populations. Studies in animal models suggest that there may be a cause-and-effect relation between the bacteria within the gastrointestinal tract and human health (811 ). Many studies demonstrate that an alteration in the composition of the human gut microbiota, its genome (referred to as the metagenome), and its metabolites (the metabolome) are associated with disease states; however, there are fewer studies suggesting that these features can be used as a biomarker to predict the future development of disease. Analyses of samples matched to clinical outcomes have suggested that the gut microbiome might have predictive value in the treatment of inflammatory bowel disease (11 ) or in the prevention of cardiovascular disease (9 ), type 2 diabetes (8 , 10 ), and metabolic syndrome (10 ). However, prospective studies are needed to validate the predictive value of these biomarkers. In metabolic syndrome, the most consistent feature associated with a disease state was decreased gut microbial richness; less microbial richness is associated with dietspecifically lower fiber and higher fat intakeand residence in an industrialized nation (10 ). There has been much interest in the impact of diet on the gut microbiota where effect sizes seem to be personalized and relatively modest (12 , 13 ). Nevertheless, the high-dimensional nature of microbiota features may be helpful in the development of personalized dietary interventions to prevent and/or treat disease (14 ). Using a case-control sample from a prospective cohort of pregnant women, we sought to determine whether, in the context of a high-fat, low-fiber diet, maternal fecal microbial diversity and metabolome during the second trimester contained features associated with future development of spontaneous PTB (SPTB). We hypothesized that a processed, high-fat, low-fiber diet would be associated with alterations in the gut microbiota, fecal metabolome, and plasma metabolome that could provide a metabolomic signature of eventual SPTB. Methods Sample In accordance with an Institutional Review Boardapproved protocol, informed consent was obtained from 301 primiparous women who were enrolled into the study at 2026 weeks of gestation. The study participants received prenatal care from 1 of 2 hospital-affiliated obstetric practices, delivered at the same hospital, and remained enrolled in this observational study until after the delivery of their infant. The key exclusion criteria were women with multiple gestation, fetal chromosomal abnormality or major fetal anomaly, intrauterine fetal demise, maternal HIV, chronic kidney disease, transplantation, history of weight loss surgery, prior completed pregnancy, vegan diet, and chronic use of immunosuppressive medications or steroids. The study design included a cross-sectional survey of diet, the fecal microbiome, and the fecal and plasma metabolome at 2026 weeks of gestation, followed by prospective observation for the primary clinical outcome of spontaneous preterm compared with term delivery. The diagnosis of SPTB was obtained from medical record review, using American College of Obstetricians and Gynecologists definitions (15 ) and confirmed by a perinatologist. Gestational age at delivery was determined by best obstetric dating based on last menstrual period and confirmed by prenatal ultrasound. An embedded case-control study (Supplemental Figure 1 ) was constructed by matching all women who underwent SPTB (cases, n = 16) to 2 women who delivered at term without complications such as gestational diabetes, gestational hypertension, or pre-eclampsia (controls, n = 32). Because of the recognized greater risk of PTB in black and obese women, cases were matched to controls by race (black/nonblack) and prepregnancy BMI group (obese/nonobese). Racial-ethnic identity was determined by self-report. Owing to the small number of women who identified as Asian or other, they were included with Caucasians to form a nonblack category for comparison with women who self-identified as black. Prepregnancy BMI was calculated from height and measured weight at the initial prenatal visit (in kg/m2 ). According to CDC guidelines (16 ), BMI 30 was used to categorize obesity. Gestational weight gain was calculated as the maternal weight at the last prenatal visit before delivery minus the weight at the first prenatal visit. Three 24-h dietary recalls were obtained during weeks 2026 of gestation via unannounced telephone calls within a 10-d interval before collection of the stool sample. A trained research nutritionist administered the recalls using the validated multipass method supported by Nutrition Data System for Research software (University of Minnesota Nutrition Coordinating Center) (17 ). Oral prenatal vitamin content was not included in the food recalls, but all patients reported taking these supplements. Food recall data were cleaned for implausible energy intake when energy intake was <600 or >3500 kcal/d (18 ). This excluded diet data in 1 case, where the energy intake was reported as 4801 kcal/d, 76 kcal kg body weight1 d1 . After the third dietary recall was obtained, the study parti-cipants collected their next stool sample and shipped it to the laboratory. During the following obstetric clinic visit, a sample of 50 mL blood was drawn and processed to obtain plasma. Plasma and fecal samples were stored at 80C until further analysis. The study protocol did not allow missing data. Study participants were then monitored prospectively for the timing of their delivery. Of the original 301 enrolled subjects, 4 delivered their infants in a different hospital that did not permit exact tracking of birth timing, and these subjects were excluded from the analyses. The stool samples were assayed for gut microbial identity using 16S tagged gene sequencing processed by Quantitative Insights Into Microbial Ecology (QIIME) 2, including de-multiplexing, quality control, amplicon sequence variant identification, and taxonomic assignment. The maternal plasma and fecal water samples were assayed for untargeted metabolomics by Metabolon (Metabolon, Inc.). Statistical and bioinformatics analysis 16S tagged sequencing data analysis The relative abundance per sample was summarized at the genus level. Richness, Shannon indexes, and Menhinick indexes were calculated to represent the microbiomediversity. The weighted (proportional abundance) and unweighted (presence-absence) Unifrac distances were calculated to represent the microbiomediversity. R package vegan was used for anddiversity calculation. Two control samples had very low read counts and were removed in the analyses that were related to microbiome data. The mean number of reads per sample ( SD) that could be assigned to the genus level was 30, . Differential abundance analysis Wilcoxons rank-sum test was used for the comparison of microbiome genera,diversity, and stool and plasma metabolites between term and SPTB. Differences in dietary intake variables were compared between term and SPTB cases using a 2-tailed t test with unequal variance. Permutational multivariate analysis of variance (PERMANOVA) was applied for the comparison ofdiversity between term and SPTB. A false discovery rate (FDR) of was used to select the bacterial genera or metabolites that were associated with SPTB. All analyses were performed using R version (R Foundation). For our given sample size, we had a power of 80% to detect an effect size (as measured by Cohens d ) of at an level of for the primary outcome of microbial diversity. For other secondary outcomes, owing to multiple comparison adjustments, only associations with a very large effect size could be detected. Weighted Gene Co-expression Network Analysis metabolite module For the stool metabolites, Weighted Gene Co-expression Network Analysis (WGCNA) (19 ) was applied to 1 ) identify functional modules of stool metabolites based on pairwise correlations, 2 ) correlate the modules with SPTB based on principal component analysis, and 3 ) find the most important metabolites within the module of interest. Such a network analysis accounts for high correlations among certain metabolites, provides effective dimension reduction, and leads to more biologically interpretable results. By using WGCNA, the correlations raised to a certain power (adjacencies) were calculated between all pairs of metabolites, so that the degree distribution would fit a scale-free model. The adjacencies were then transformed into topological overlap measures (TOMs) and TOM-based distance matrices. Finally, a dynamic branch-cutting method was applied to detect clusters (modules) of metabolites depending on the shape of the clustering tree. Pearsons correlation was calculated between each module (represented by the first principal component) and SPTB. The important metabolites in each module were defined as metabolites with high metabolite significance (represented by absolute Pearsons correlation with SPTB > ) and high intramodular connectivity (represented by absolute Pearsons correlation with the first principal component of the module > ). R package WGCNA was used for the analysis. Results Baseline characteristics From a prospective cohort observation of 301 pregnant women, 16 women delivered spontaneously earlier than 37 weeks of gestation (Supplemental Figure 1 ). Four women were considered to have early SPTB (2430 wk) and 12 were categorized as late SPTB (3136 wk). All cases, including 8 black and 8 nonblack women (6 white and 2 Asian), were matched with 2 controls of the same self-reported race and obesity status (18 obese and 30 nonobese women). As expected, owing to the preterm delivery, gestational weight gain and gestational age at delivery were lower in cases, but gestational weight gain per week was similar for both groups ( Table 1 ).Diversity of fecal microbiota was reduced in women with SPTB In order to determine the difference of diversity at the genus level in the fecal microbiota, 3 measures ofdiversity were evaluated: 1 ) richness, the total number of different genera in a sample; 2 ) Shannon diversity, which accounts for the proportion (evenness) of each genus in a sample; and 3 ) Menhinicks index, the number of genera divided by the square root of the total number of individuals. The 3diversity indexes were all significantly reduced in fecal microbiota collected between 20 and 26 weeks of gestation from women who ultimately had SPTB ( Figure 1 A , Shannon diversity P = Figure 1B , richness P = Figure 1C , Menhinicks index P = ). To determine the taxonomic features responsible for this difference, Shannon diversity was calculated for each bacterial class ( Figure 2 A ). The bacterial class of Betaproteobacteria was identified as showing a significant reduction in Shannon diversity in SPTB cases (Wilcoxons rank-sum test, FDR < ) (Figure 2B ), where the AUC on logistic regression analysis was (Figure 2C ). No specific genera in Betaproteobacteria showed a statistically significant difference in relative abundance in cases compared with controls (Wilcoxons rank-sum at FDR = ) (Supplemental Figure 2 ). Owing to the small sample size, to test for predictive performance using within-class diversity, we performed leave-one-out cross-validation analysis. The resulting AUC of Betaproteobacteria was (Supplemental Figure 3 ).Diversity of the fecal microbiome differed in mothers who had SPTB compared with full-term birth Both weighted and unweighted Unifrac distance were significantly different between cases and controls (PERMANOVA, p= for weighted analysis, Figure 3 A ; p= for unweighted analysis, Figure 3B ). We were unable to identify any statistically significant discriminatory taxa based on the relative abundances at the genus level (106 genera tested) at an FDR of . SPTB was associated with an increase in 2 lipid metabolites in the fecal metabolome To identify additional analytic features associated with SPTB, we performed untargeted metabolomics analysis on maternal fecal and plasma samples collected at the prenatal visit. Out of 824 fecal metabolites, 22 were different in abundance between cases and controls (P < , Wilcoxons rank-sum test). The majority of fecal metabolite differences were found among fatty acids and cholesterol hormone metabolites ( Figure 4 A ). After correction for multiple comparisons (FDR < ), both DHA (22:6n3) and EPA (20:5n3) were increased in the fecal metabolites of women with SPTB (Figure 4A ). Similarly, out of 825 plasma metabolites, 4 [-citrylglutamate, propionylglycine, caproic acid (6:0), and 14-hydroxydocosahexanoic acid (HDoHE)/17-HDoHE] were different in abundance (P ), although none were significant after correction for multiple comparisons (Supplemental Table 1 ). Cases with SPTB had greater intake of dietary fatboth saturated and polyunsaturatedthan women who delivered at term Given the previously described associations between dietary intake and the gut microbiota (12 , 13 , 20 ), we used the mean of three 24-h dietary recalls administered before collection of the stool sample to characterize the average diet of cases and controls ( Table 2 ). Women with SPTB had greater intake of dietary fat ( g compared with g, P = ) with a trend toward a higher percentage of calories from fat ( compared with , P = however, both groups were close to the upper limit of the reference range for dietary fat intake based on the American Heart Association recommendations (20%35% of calories from fat). The intake of saturated fat was higher in cases ( g compared with g, P = ), predominantly palmitic acid (16:0) ( g compared with g, P = ). There was no difference in intake of monounsaturated fat. PUFA intake was greater in women with SPTB ( g compared with g, P = 9% compared with 8% kcal, P = ). Both omega ()-3 fatty acids ( g compared with g, P = ) and6 fatty acids ( g compared with g, P = ) were higher in mothers with SPTB. Both groups had a ratio of6 to3 higher than the recommended ratio of <4:1, which is consistent with the overall poor quality of diet in this cohort of women ( for cases compared with for controls, P = ). All women reported taking prenatal vitamins that included 200 mg DHA. However, both groups had < g/d of the beneficial long-chain3 fatty acids, DHA and EPA, from their diet. In contrast, both cases and controls met the minimum DRI ( g/d during pregnancy) of the precursor3 fatty acid found in seed oils,linolenic acid (18:3n3), with cases ingesting significantly morelinolenic acid than controls ( g compared with g, P = ). Intakes of6 linoleic acid (18:2n6) ( g compared with g, P = ) and transoctadecadienoic acid (18:2n6) ( g compared with g, P = ) were also higher in cases; these fatty acids are predominantly found in processed vegetable and seed oils. Consistent with higher intake of vegetable and seed oils, cases also had higher amounts oftocopherol ( mg compared with mg, P = ) and a trend toward higher amounts oftocopherol ( mg compared with mg, P = ) (21 ). We used partial Kendall correlation to examine the associations between the statistically significant dietary intake variables and fecal metabolites, which can be visualized in a heatmap (Figure 4B ). Total saturated fat intake, the majority of which was palmitic acid, was positively associated with fecal metabolite concentrations of DHA, EPA, and docosapentaenoic acid (DPA; 22:5n3) (each P < ), as well as sugars (glucose, ribulonate/xylulonate/lyxonate), cholesterol breakdown (durobilin, ursocholate), and steroid hormone metabolites (pregnanolone/allopregnanolone sulfate) (each P < ). Total PUFA intake, including both linolenic and linoleic acid, was positively associated with fecal concentrations of ursocholate, dehydroepiandrosterone sulfate (DHEA-S), and pregnanolone/allopregnanolone sulfate (all P < ). These correlations provide objective analytic measures that parallel the dietary information collected by self-reported recall. Four modules in the fecal metabolome were associated with SPTB After individual tests on single metabolites, we used WGCNA to identify and evaluate whether specific metabolic modules or pathways were correlated with SPTB. A total of 20 modules were recovered from stool metabolomics data, and the module-level correlation analysis identified 4 modules highly correlated with SPTB (Supplemental Figure 4 , Pearson correlation value > P value < ). The modules represented as connected graphs of metabolites are shown in Figure 4C and Supplemental Figures 57 , where 2 metabolites are connected if the edge weight was >. To summarize the metabolite measures within a module, the first principal component was calculated and was associated with SPTB. Members of the 4 most strongly associated modules included most of the differentially abundant metabolites identified from the individual variable tests (Wilcoxons rank-sum, P value < ). The 4 most associated modules included closely related metabolites that are similar in chemical structure and/or closely related in biological function. The module most strongly associated with SPTB included 26 metabolites and was enriched in3 fats (Figure 4C , Supplemental Table 2 ). Three other modules with P value < are in Supplemental Figures 57 and Supplemental Table 2 . One module (Supplemental Figure 5 , P value = , FDR-adjusted P = ) contained a high representation of steroid hormones, including bile acids and DHEA-S for which elevated concentrations have recently been associated with preterm labor (22 ). Another module (Supplemental Figure 6 , P value = , FDR-adjusted P = ) was most strongly represented by ceramide and sphingosine signaling molecules that are recognized as proinflammatory. Further, sphingosine-1-phosphate has been identified for its role in myometrial contraction in response to LPS-induced infection (23 ), thus highlighting the potential role these modules may play in the etiology of SPTB. The module including sugars, fats, and alcohol was associated with SPTB overall (Supplemental Figure 7 , P value = , FDR-adjusted P = ), although no single member of the module was associated with SPTB with a Pearsons correlation value > . Discussion In this prospective, case-control study of pregnant women matched by nationally recognized variables associated with SPTB, we identified features of the fecal microbiota and metabolome that discriminate between women who eventually develop SPTB and those who carry their pregnancy to term. A decrease indiversity was strongly associated with the development of SPTB where a reduction in the taxonomic class of Betaproteobacteria performed moderately well in discriminating SPTB. Whereas there were no features in the maternal plasma metabolome associated with the development of SPTB after adjusting significance levels for FDR, the fecal metabolome had distinct increases in excretion of lipid and steroid hormone metabolitesspecifically DHA, EPA, DHEA-S, and allopregnanolonethat were associated with greater dietary fat intake (especially SFAs, PUFAs, and trans fatty acids). These results not only suggest that the composition of the gut microbiota may ultimately have utility in predicting SPTB, but, together with the fecal metabolome, may also be congruent with previous epidemiologic studies showing that the consumption of a high-fat, low-fiber diet is associated with an increased risk of SPTB (5 , 7 , 2426 ). Few studies have examined the association between diet and the composition of the gut microbiota during pregnancy. In 2 studies of the same cohort of Norwegian pregnant women, higher dietary fiber and moderate fat intake were associated with increased microbial richness and fewer Bacteroidaceae (27 ), whereas higher intake of monounsaturated fat, cholesterol, and fat-soluble vitamins was associated with increased potentially proinflammatory bacteria of the genus Proteobacteria (28 ). Another study reported reduced Shannon diversity in postpartum fecal samples from women delivering preterm but did not report dietary intake or metabolomics data (29 ). Our study is the first that we know of to describe maternal gut microbiota and fecal metabolomics-based features that are associated with SPTB in a prospective cohort and to identify reduced microbial diversity in Betaproteobacteria. Although most Proteobacterial bacterial taxa associated with disease states belong to the class of Gammaproteobacteria, Betaproteobacteria taxa have also been associated with disease. For example, Alcaligenes spp. have been associated with inflammatory diseases in mice and humans (30 ) and both Sutterella and Parasutterella have been associated with both diet and disease (31 , 32 ). We also noted weaker associations with plasma metabolites with potential relevance during pregnancy. 14-HDoHE is a lipoxygenase metabolite of DHA recognized for low activation of platelets (33 ). 17-HDoHE is a DHA-derived oxylipin signaling molecule that has been linked with reduced oxidative stress and oxidative damage in hepatocytes, and suggested as a possible novel biomarker of surgical systemic inflammatory stress (34 ). It has also been suggested to be a proresolving mediator and peroxisome proliferatoractivated receptor-/ agonist in cell culture (35 ). In pregnancy, the concentration of plasma oxylipins and precursor fatty acids has been suggested as a potential predictor of SPTB (36 ). In the current study, the ratio of 14-HDoHE to 17-HDoHE was increased in women with SPTB; however, the meaning of this difference during pregnancy is not yet clear. Importantly, these analytic features were consistent with the subtle differences in diet between cases and controls based on dietary recall data. Numerous researchers have shown that diet has significant influence on the diversity and richness of the gut microbiota (12 , 20 , 3743 ). Greater intake of processed foods and proinflammatory fatty acids (reflected by higher SFAs,6 PUFAs, and trans fatty acids) has been linked to obesity, diabetes, and cardiovascular disease, yet less is known about the impact of greater intake of processed and inflammatory fats during pregnancy. Large international cohort observations have linked a high-fat, low-fiber diet with increased risk of SPTB and a lower-fat, higher-fiber diet with reduced risk (7 , 2426 ). Similarly, highly processed, high-fat diets that are low in fiber are linked with decreased microbiota diversity, whereas those rich in fiber but low in fat are associated with increased diversity (12 , 20 ). The diet in this study might be described as a comparison of a high-fat, low-fiber diet in controls with a yet worse quality diet in cases. Both were poor in quality but the diet in cases was worse and associated with SPTB in our cohort. Although fiber intake was low in both cases and controls, the predominant dietary difference was greater fat intake (especially SFAs, PUFAs, and trans fatty acids) in women with SPTB. The analytic data on the composition of the gut microbiota, together with 2 features in the fecal metabolome, were congruent with consumption of a poor-quality diet that includes higher amounts of proinflammatory dietary fat in mothers at risk of SPTB. The dietary recall data in our study were suggestive of higher intake of plant and seed oils, including significantly greater intake of PUFAspredominantly6as well as highertocopherol in processed foods. This signature was associated with higher maternal fecal concentrations of the steroid hormone metabolite DHEA-S. Both DHEA-S and allopregnanolone are fetal steroid molecules derived from breakdown of cholesterol that may signal intrauterine fetal stress. Intriguingly, maternal serum DHEA-S concentrations have been identified as one of the signals for cervical ripening (44 ) and indicators of successful induction of labor in term pregnancies (45 ). Similarly, the higher concentrations of fecal EPA and DHA paired with a decrease indiversity of the gut microbiota seen in women with SPTB were congruent with the dietary recall data showing that these women were consuming a higher-fat diet than controls. In addition, several bile acid metabolites (durobilin, lurobilinogen, and ursocholate) cosegregated in the fecal metabolomics analysis, a finding that would be expected with greater dietary fat intake and a subsequent increase in biliary secretions (Figure 4A , B ). Trans fatty acids are associated with cardiovascular disease and proinflammatory cytokine production by toll like receptor-4 activation, with the potential for deleterious effects on the fetus (46 ). Trans fatty acids have also been suggested as a risk factor for inadequate placental delivery of3 DHA to the fetus. Trans fatty acids are inversely related to DHA and arachidonic acid (20:4n6) concentrations in maternal plasma during pregnancy and at delivery and in fetal cord blood (47 , 48 ), suggesting possible displacement by trans fatty acids of essential fatty acids in the case of placental transfer of this essential fatty acid for brain development to the fetus. In contrast to women in this cohort who delivered at term, trans fat intake in cases ( g/d) exceeded the recommended intake amounts of <2 g/d and was also considerably greater than current Danish intake amounts (<1 g/d) where the rates of SPTB are lower than in the United States (48 ). This study is limited by its small sample size, with only 16 cases of SPTB (5%). By contrast, national data include cases of medically indicated PTB due to pregnancy complications and women with a history of SPTB in prior pregnancies, both excluded from this study. However, this sample is an unusually rich representation of black women in a northeastern urban setting, a segment with limited previous description of diet and the gut microbiome during pregnancy. Even though black women have been recognized as at higher risk of SPTB (3 ), studies identifying novel predictive variables are limited, thus the current study is important. The limited generalizability of these findings to other samples with different racial and ethnic diversity or rural settings will require further study. Nonetheless, the ability to validate self-reported dietary recall data by using independently determined quantifiable analytic features adds to the strength of our findings. It is also important to consider that both cases and controls had poor-quality, highly processed high-fat, low-fiber diets, so our ability to detect a dietary effect should have been modest. Yet, we observed differences in gut microbiota diversity and fecal metabolites. Future studies are needed comparing pregnant women eating a minimally processed healthy diet with those consuming a highly processed unhealthy diet to more adequately determine the potential impact of diet quality on the gut microbiota and birth timing during pregnancy. In conclusion, data obtained several weeks before delivery suggest that reduceddiversity of the fecal microbiota and increased fecal concentration of lipid (DHA, EPA, DPA) and steroid (DHEA-S and allopregnanolone) metabolites might provide a biomarker signature associated with future SPTB in women consuming a high-fat, low-fiber diet. Further investigation of these parameters in a larger sample is needed to evaluate the reproducibility of the findings. Translational studies in a mouse model of SPTB also might enlighten and enable full assessment of the absorption dynamics and any role that tissue concentrations of3 fatty acids might play. The authors responsibilities were as followsCWC, GDW, and ME: designed the research; CWC and ME: conducted the research; CWC, HL, YL, and GDW: analyzed the data or performed statistical analysis; CWC, GDW, YL, and VG: wrote the manuscript; CWC and GDW: had primary responsibility for the final content; and all authors: read and approved the final manuscript. The authors report no conflicts of interest. Data Availability Data described in the article, code book, and analytic code will be made publicly and freely available without restriction at . SUPPORTING INFORMATION nqaa361_Supplemental_Files\",\n          \"INTRODUCTION: Low gut microbiome richness is associated with dyslipidemia and insulin resistance, and ceramides and other sphingolipids are implicated in the development of diabetes. OBJECTIVES: Determine whether circulating sphingolipids, particularly ceramides, are associated with alterations in the gut microbiome among obese patients with increased diabetes risk. METHODS: This was a cross-sectional and longitudinal retrospective analysis of a dietary/weight loss intervention. Fasted serum was collected from 49 participants (41 women) and analyzed by HPLC-MS/MS to quantify 45 sphingolipids. Shotgun metagenomic sequencing of stool was performed to profile the gut microbiome. RESULTS: Confirming the link to deteriorated glucose homeostasis, serum ceramides were positively correlated with fasting glucose, but inversely correlated with fasting and OGTT-derived measures of insulin sensitivity and beta-cell function. Significant associations with gut dysbiosis were demonstrated, with SM and ceramides being inversely correlated with gene richness. Ceramides with fatty acid chain lengths of 20-24 carbons were the most associated with low richness. Diet-induced weight loss, which improved gene richness, decreased most sphingolipids. Thirty-one MGS, mostly corresponding to unidentified bacteria species, were inversely correlated with ceramides, including a number of Bifidobacterium and Methanobrevibacter smithii. Higher ceramide levels were also associated with increased metagenomic modules for lipopolysaccharide synthesis and flagellan synthesis, two pathogen-associated molecular patterns, and decreased enrichment of genes involved in methanogenesis and bile acid metabolism. CONCLUSION: This study identifies an association between gut microbiota richness, ceramides, and diabetes risk in overweight/obese humans, and suggests that the gut microbiota may contribute to dysregulation of lipid metabolism in metabolic disorders. Type 2 diabetes is the result of insulin resistance and deterioration of \\u03b2-cell function. Yet the environmental factors, and their links to physiological mediators, that drive the development of type 2 diabetes are only partially understood. The gut microbiota has emerged as a potential contributor to the regulation of glucose homeostasis. Type 2 diabetes is associated with alterations in a number of bacteria and their biological functions in both Chinese () and European () populations. Metabolic changes that precede frank diabetes can also be transferred via the gut microbiota to mice (). Low gut microbiota richness, which is altered in some moderate obesity/overweight patients without overt diabetes (), may be an important factor in the progression to diabetes. The microbial ecosystem can be characterized by its richness . the number of unique bacterial genes present in a metagenome. People in Western societies, with high rates of obesity, type 2 diabetes, and frequent exposure to antibiotics and food additives that can impact gut microbes, have considerably lower bacterial diversity than previously unexposed hunter-gatherers (). Gastrointestinal diseases such as liver cirrhosis () and inflammatory bowel diseases () have been associated with a loss of metagenomic richness. Metabolic characteristics are also altered in individuals with low gut microbiome richness, as Danish subjects with fewer microbial genes have increased BMI, insulin resistance, inflammatory markers, as well as hypertriglyceridemia (). These associations were also observed in French individuals and our laboratory further showed that this was partially reverses by diet-induced weight loss (). Lipid metabolism is critical to the pathogenesis of diabetes () and alterations in the gut microbiota could potentially regulate these pathways. Sphingolipids are a diverse class of membrane lipids that also act as intracellular signaling molecules. Ceramides, which accumulate with lipid loading and inflammation, are increased in patients with type 2 diabetes (; ), and they can directly induce insulin resistance () and regulate \\u03b2-cell apoptosis (). Plasma ceramides may even become clinical biomarkers, but the environmental factors that contribute to ceramide accumulation, outside of obesity, are not well defined (). It was recently shown in mice, however, that the gut microbiota regulates liver ceramide synthesis and subsequent hepatic steatosis and gluconeogenesis through FXR-dependent secondary bile acid metabolism (). Furthermore, endotoxin-induced inflammation, a potentially important link between impaired glucose homeostasis and gut dysbiosis (), readily stimulates sphingolipid synthesis. It is still unknown, however, whether an altered gut microbiota is associated with increased ceramide levels in humans. Combining targeted lipidomics and quantitative metagenomics, the objective of the present study was to examine whether ceramide accumulation could link gut dysbiosis to increased diabetes risk in individuals with overweight or obesity. The study was conducted at Pitie\\u0301-Salpe\\u0302trie\\u0300re Hospital (Nutrition department) in Paris, France. Forty-nine participants were overweight (N=11) and obese (N=38) (8 males, 41 females), and have been described in detail elsewhere (; ). Participants did not have diabetes, chronic or inflammatory diseases, or related treatments, and did not take antibiotics within the two months prior to stool collection. The intervention consisted of a 6-week period on a low calorie diet enriched with soluble fibers, high protein intake, and low glycemic index carbohydrates. Daily caloric intake was approximately 1,200 kcal/d for women and 1,500 kcal/d for men, composed of 35% protein, 25% fat and 40% carbohydrate. A 6-week weight stabilization diet followed the calorie restriction period, and was established based on each subject\\u2019s resting energy expenditure and physical activity levels. Details of the analyses have been described elsewhere (; ). Body composition was determined using dual energy x-ray absorptiometry for the estimation of total body fat and android fat proportion. Subcutaneous adipose tissue samples were evaluated for adipocyte size and immune cell frequency (). Blood samples were collected after an overnight fast to measure routine biochemical parameters. Serum insulin was assayed by a chemiluminescense (ARCHITECT system, Abbott). Insulin sensitivity was quantified using the following fasting indices: the Homeostasis Model Assessment of insulin sensitivity (HOMA-S), calculated using the HOMA2Calculator system (); Quantitative Insulin Sensitivity Check Index (QUICKI) (); and the Disse index (), a lipid-based index of insulin sensitivity An oral glucose tolerance test (OGTT) was performed in 40 subjects (39 that had insulin measured during the test at all time points). The Matsuda Index was calculated as a measure of insulin sensitivity (). \\u03b2-cell function was estimated by the insulin secretion and sensitivity index-2 (ISSI-2), an OGTT-analog of the IVGTT-derived disposition index. It is the product of insulin secretion in response to glucose, measured as the ratio of insulin AUC to the glucose AUC, and the Matsuda index. The ISSI-2 captures the hyperbolic relationship between insulin sensitivity and secretion () and correlates with the disposition index (). We confirmed the hyperbolic relationship in our sample population, as outlined in the original paper (data not shown) (). Analysis of sphingolipids was conducted by HPLC-MS/MS, as described previously (), in overnight fasted serum. Lipids were extracted in acidified methanol:chloroform with both internal and calibration standards (Avanti Polar Lipids, Alabaster, AL, USA). Forty-five sphingolipids were quantified, namely sphingomyelins (SM), dihydroceramides (DHCer), defined as Cer(d18:0) in standard nomenclature, and sphingosine- or sphingadienine-containing ceramides, that is Cer(d18:1) and Cer(d18:2), respectively. These 45 lipids were detected in at least 80% of samples. To avoid bias introduced by simple 0-imputation of the values below detection, measurements below the level of quantitation were imputed using multiplicative log-normal-randomized imputation with the zCompositions package in R. Total faecal DNA was extracted and sequenced on a high-throughput ABI SOLiD sequencer. Reads were cleaned for quality and origin (human, cow, and plant), mapped to the million gene reference catalogue () and normalized by nucleotide length. Forty-five subjects with sufficient sampling depth were used for subsequent analyses. Gene richness was calculated, after downsizing to 7 million genes, as the total number of uniquely mapped genes per sample. Richness categories were split at 460,000 genes in the catalog, as done previously (). Metagenomic species (MGS) were constructed from total-sum normalized gene abundance data using the catalog provided by Nielsen  and only co-abundance clusters greater than 700 genes were considered as bacterial species (). Computations were performed using the [momr] R package (; ) Kegg Ortholog (KO) annotation was provided by the reference catalog, and each gene was summed into its respective KO. Annotation of bile acid genes was performed as described by others (). Briefly, protein sequences from organisms containing complete bile acid pathways were extracted, specifically: Clostridium scindens ATCC 35704, Bifidobacterium longum SBT2928, Bacteroides fragilis NCTC9343, Eggerthella lenta DSM 2243, and Ruminococcus gnavus ATCC 29149. Sequences were then aligned to the reference catalog using UBLAST with an E-value cutoff of 10. These candidates were then aligned against the KEGG database to include only genes with best hits to bile acid proteins. Bile acid genes were grouped by their functional annotation in MetaCyc (. bile acid degradation). Statistical analyses were performed in the R Language and Environment with packages listed in [brackets]. As the data were generally positively skewed (lipidomics) or non-normal (metagenomics), robust statistics were used when possible. The Spearman method was used for correlations. The MM-type robust regression [lmrob function in robustbase] was used for multiple regression (). Longitudinal data were analyzed using mixed effects models with random intercepts for subject [lme4, car, multcomp]. Enrichment of KEGG modules and pathways were tested with the reporter features algorithm using adjusted p-values and coefficients from the correlations of each KO to Cer(d18:1) [piano] (). Only distinctly up- and down-regulated gene sets are reported. The null distribution was used for the significant assessment of gene sets between 3 and 100 KOs. For clinical (including gene richness) and lipid class associations, unadjusted p-values were reported. However, given the a priori nature of metagenomic analysis, p-values for tests of MGS or KOs was adjusted for the Benjamini-Hochberg false discovery rate; the only exception being bile acid gene set enrichment, as this hypothesis was pre-specified (). Characteristics of the population used for lipidomics profiling have been described previously (), and are summarized in and and . Median BMI was approximately 33 kg/m and body fat was 40%. None of the subjects had diabetes. Shotgun metagenomic analysis indicated that 41 women had a median gene richness of 507,800 and 8 men had a median gene richness of 537,000, which were not statistically significantly different. We confirmed that gene richness was inversely associated with fasting TG (r=\\u2212, P<), as previously reported. Serum sphingolipids were tested for associations with a number of diabetes risk factors. Four sphingolipid subclasses were quantified, covering a total number of 45 lipids including sphingomyelins (SM), dihydroceramides (DHCer) or Cer(d18:0) in standard nomenclature, and (d18:1)Cer or (d18:2)Cer ceramides, which correspond to a classical sphingosine backbone or a monounsaturated variation, respectively. The most significant associations found were with the non-SM subclasses and clinical variables linked to deteriorated metabolic homeostasis such as accumulation of android fat, age, cholesterol, triglycerides, and GGT (). Importantly, regarding glucose homeostasis, all subclasses were positively associated with 60 min glucose during the OGTT, but only two subclasses, DHCer and Cer(d18:1), were associated with impaired insulin sensitivity determined by both fasting (Disse, QUICKI) and the OGTT-derived Matsuda index (). Furthermore DHCer, Cer(d18:1), and Cer(d18:2), inversely associated with the ISSI-2 index, indicative of impaired \\u03b2-cell function (). SM, Cer(d18:1) and Cer(d18:2) were also positively correlated with circulating CRP or IL-6. In order evaluate the relative weight of each sphingolipid subclass in the association with clinical parameters, the sum of the number of variables (excluding cholesterol and TG) with significant associations with individual shphingolipid species was calculated. This revealed that DHCer and Cer(d18:1) were more strongly associated with clinical parameters than other species; together they accounted for 7 of the top 9 lipids with the highest number of non-lipid (. HDL-C) clinical correlations (; ). Importantly, very-long chain ceramides (C22 \\u2013 C24 species), which are the most abundant species in human serum (), were as related to clinical phenotype as C16 and C18 species. Overall, these data confirm that serum ceramides levels are associated to a deleterious metabolic profile. Sphingolipids may be an important component of the dyslipidemia associated with low bacterial richness. Consistent with this hypothesis, there was an inverse association between gene richness and each lipid subclass, although the statistical significance was strongest for Cer(d18:1) (rho=\\u2212, P=) and Cer(d18:2) (rho=\\u2212, P=) (, ). In multivariable analysis adjusting for age, sex, and fasting triglycerides, SM was no longer significant at P<, while Cer(d18:1), Cer(d18:2), and to a lesser extent DHCer, remained statistically significant (). At the level of individual sphingolipids species, the relationships between SM species were generally attenuated compared to the Cer(d18:1) and Cer(d18:2) (). After adjustment, the strongest associations were among species containing between 20 and 24 carbons in Cer(d18:1) or Cer(d18:2) (). These data demonstrate a significant link between serum ceramides and gut microbiota richness, and reveal unexpected importance of long chain ceramides in this association. Longitudinal changes in gene richness would bolster the cross-sectional findings. As reported previously (), diet-induced weight loss significantly increased gene richness in subjects with low gene richness before the intervention (). We therefore examined if weight loss also corresponded to changes in circulating sphingolipids. DHCer, Cer(d18:2), Cer(d18:1), and SM were decreased by 47%, 39%, 35%, and 26%, respectively, after a 6-week period of diet restriction in the obese subjects (all P<), and remained suppressed\\u2014with some rebound\\u2014during the 6 additional weeks of weight stabilization (). Over the weight loss period, few changes were seen in patients with high gene richness at baseline, whereas whose with initial low gene counts ameliorated their low microbiome diversity (). Regression of lagged variables confirmed a longitudinal association: an increase in gene richness from the previous time point was associated with a concomitant decrease in sphingolipid subclasses among participants with low baseline richness (). Overall, these data demonstrate association between blood ceramide levels and dynamic gut microbiota status driven by diet interventions in subjects with overweight/obesity. We next investigated if some specific bacterial species could be related to host ceramide levels. Lower gene richness was associated with a decrease in 97 of the 310 MGS (Padj< ), consistent with the loss of bacterial richness. Firmicutes, the most abundant phylum in the human intestine, comprised the majority of associations with both gene richness and sphingolipids (). Decreased abundance of 30 MGS were associated with an increase serum concentrations of Cer(d18:1); approximately half of these bacteria (14 MGS) also showed significant association with Cer(d18:2) in the same direction (). A number of the sphingolipid-related MGS could be annotated to the species level. In particular, Methanobrevibacter smithii, Bifidobacterium adolescentis, Alderceutzia equolifaciens, Ruminococcus obeum, and Eubacteria hallii were associated with both low gene richness and high serum levels of Cer(d18:1). Alistipes indistinctus, Dialister invisus, and Bifidobacterium bifidum were also negatively associated with Cer(d18:1), but not with gene richness. Only one MGS was inversely associated with DHCer and one with SM. Ruminococcus gnavus was positively associated with Cer(d18:1). Thus from this analysis, it appears that the link between serum ceramide levels and gut microbe communities mainly relate to the loss of bacterial subgroups. Quantitative metagenomics allows direct characterization of the functional potential of the microbiota. Consistent with the MGS data, gene richness was associated with changes in a large number of orthologue genes grouped within KO annotations (approximately half of total KOs detected). Interestingly, serum levels of Cer(d18:1) and Cer(d18:2) showed significant association with approximately 20% of annotated KO, and only a small percentage related to DHCer and no KOs significantly correlated to SM (). Ninety-one percent of Cer(d18:1) and 95% of Cer(d18:2) associated KOs were also associated with gene richness (). Because studies in mice have uncovered secondary bile acid metabolism by the gut microbiota as an important regulator of ceramides (), we examined the relationship between Cer(d18:1) and bacterial genes involved in bile acid transformation. Genes involved in glycoholate metabolism, bai genes coding the 7\\u03b1-dehydroxylation pathway for secondary bile acid synthesis (bile acid degradation), and iso-bile acid synthesis genes were found with a decreased frequency with higher host serum ceramides levels (Padj<), which suggest different potential for bile acid transformation in subjects with elevated ceramides. Analysis at the module level confirmed the inverse association with bile acid metabolism genes (p=), and bile acid degradation (P=) more specifically (). Longitudinal data confirmed these findings since patients with low richness at baseline exhibited an increase in abundance of several bile acid genes, including bsh, during the dietary intervention concomitant with the increase in richness and decrease in ceramides (). A hypothesis-free analysis of all available KEGG modules and pathways was also performed at baseline. At the module level (), the strongest inverse association with serum Cer(d18:1) were related to methanogenesis and cellular processes comprised of archaea-specific enzymes (RNA polymerase and archaeal proteasome). Lower capacity for nutrient transport including branched-chain amino acids and polar amino acids were a notable signature bacterial communities with elevated host Cer(d18:1). At the pathway level (), lower abundance of general cellular processes, metabolism of alanine, and carotenoid biosynthesis were detected. The majority of KEGG modules enriched with higher Cer(d18:1) were related to biosynthesis. Lipopolysaccharide biosynthesis was highly enriched at the module level, as was the biosynthesis of amino acids (tryptophan) and B-vitamins (thiamine, pantothenate). Higher capacity for LPS biosynthesis and flagellar assembly were positively related to Cer(d18:1) at the pathway level. Thus, differences in overall metagenomic richness, bacterial abundance, and functional capacity are related to host ceramides accumulation. We herein establish an association between the human gut microbiota and ceramides\\u2014lipid mediators implicated in the development of type 2 diabetes. After confirming the links between serum ceramides and diabetes risk, we found that low metagenomic richness was associated with significantly higher levels of serum ceramides. Decreased abundance of Methanobreviibacter smithii and anti-inflammatory bifidobacteria species were related to higher ceramides levels. Importantly, the ceramide-associated microbiome demonstrated an increase in a number of biosynthetic pathways, notably increased LPS synthesis and flagellar assembly. Thus, low metagenomic richness may contribute to increased diabetes risk in humans by permitting an inflammatory dysbiosis that induces ceramide accumulation. Studies in mice have implicated the gut microbiota in the regulation of host metabolism (). Gut microbiota composition is also altered in patients with type 2 diabetes from both China and Europe (; ). In the Chinese population, the individuals with type 2 diabetes had an increase in a number of pathogenic bacteria and a decrease in anti-inflammatory bacteria (). In Swedish individuals, type 2 diabetes and elevated triglycerides were primarily associated with decreased abundance of bacterial species, and their metagenomes were enriched in fructose and mannose metabolism and phosphotransferase systems, similar to our findings in subjects without type 2 diabetes. More recently, it was shown that metformin treatment at least partially confounds the relationship with type 2 diabetes (), therefore a major strength of the current study is the persistence of these findings in untreated participants. Decreased gut microbial richness is observed in overweight and obese patients, and is associated with markers of insulin resistance, systemic inflammation (. CRP), and hypertriglyceridemia (), and these factors are improved with diet-induced weight loss (). Low gene richness has also been associated with more severe conditions such as ulcerative colitis (). Thus, microbiota richness is lower with both metainflammation and chronic inflammatory diseases, which may stimulate ceramide production. In addition to lipid loading, toll-like receptor activation leads to the accumulation of ceramides (). Lipopolysaccharide, the major TLR-4 ligand, induces both de novo ceramide synthesis () and conversion of SM into ceramides via sphingomyelinases (). Endotoxin-induced cytokines also lead to ceramide accumulation (). LPS can induce ceramide accumulation in muscle and liver (), as well as lead to increased secretion of ceramides into lipoprotein particles and serum (). Chronic endotoxemia and cytokine signaling could therefore induce ceramide accumulation in obesity. Indeed, high fat feeding in mice increases circulating LPS two- to three-fold and increases the proportion of LPS-containing bacteria in the gut (). Conversely, prebiotic treatment that restores bifidobacterium species in the intestine of rodents are correlated with reduced endotoxemia and improved glucose homeostasis (). We observed a decrease in Bifidobacterium bifidum and Bifidobacerium adolescentis with higher ceramide levels, consistent with these observations in mice. Notably, Ruminococcus gnavus was positively associated with ceramides, and this species is increased in inflammatory bowel disease and appears to regulate the composition of the mucosa-associated bacterial network in these diseases (). The positive associations between increased LPS biosynthesis and flagellar assembly, another pathogen-associated molecular pattern (), corroborates the link between ceramides and gut-derived inflammation. Together, albeit only observational, our results support the hypothesis that the loss of overall diversity permits a shift towards increased endotoxin burden in the intestine, activating inflammatory pathways in the host, . ceramide accumulation. Microbial bile acid metabolism may also regulate host ceramides synthesis. Jiang  showed that the gut microbiota regulates serum ceramide levels through intestinal farsenoid-X-receptor (FXR) signaling (). Specifically, they found that reduced hydrolysis of tauro-beta-muricholic acid (by bsh), an antagonist of intestinal FXR signaling, reduces intestinal FXR activation, mitochondrial dysfunction, and circulating ceramides. In contrast, we did not detect a relationship between bsh gene abundance, and even found that reduced capacity for overall secondary bile acid metabolism was associated with higher circulating ceramides. The design of our study prohibits conclusive statements regarding the role of secondary bile acid metabolism and ceramides, but future studies incorporating careful analysis of fecal and serum bile acid levels, as well as bsh activity in stool, could help clarify this potentially important mechanism. Regarding different ceramide species, we found that longer chain Cer(d18:1), namely C20, C22, and C24 species, were the most strongly associated with both metabolic parameters and gene richness. This acyl distribution, regulated by the CERS2 gene, suggests that the gut microbiota may be particularly related to hepatic ceramide synthesis (). These results are contrary to recent genetic studies in mice that show that Cer(d18:1/16:0) is the primary driver of metabolic dysfunction (), and other reports of no effect () or even a beneficial effect of very-long chain ceramides (). Similar conclusions were also drawn from an epidemiological study of coronary artery disease (). On the other hand, nearly all circulating ceramides, regardless of fatty acids, have approximately the same degree of association with type 2 diabetes and insulin resistance (), or, as with our study, show equivalent or greater elevations in C18 to C24 ceramides compared to Cer(d18:1/16:0) (; ). Consistent with the relationships to \\u03b2-cell function in our study, circulating very-long chain DHCer predict incident type 2 diabetes (). These discrepant findings may be reconciled by a specific action of circulating ceramides. LDL-enriched in Cer(d18:1/16:0) or Cer(d18:1/24:0), the latter being far more abundant in serum, induce insulin resistance equivalently in vitro and in vivo (). Further, extracellular ceramides, derived from exogenous delivery, can enhance hepatic gluconeogenesis, consistent with the positive association with fasting glucose in our study (). Further work is therefore needed to determine the causal role, if any, of circulating ceramides, and their acyl-chain length, in the pathogenesis of insulin resistance in humans. Several limitations of our study should be noted. Most of the evidence for ceramide-induced insulin resistance comes from cellular accumulation within metabolic tissues. While we have discussed the evidence for a potential role for circulating ceramides, future studies that include both metagenomic analysis and tissue lipidomics would be an important next step. Our study is observational: while we have adjusted for a number of potentially important confounders, larger human studies and gut microbiota transfer studies would both aid in confirming the importance of the gut microbiota on ceramide metabolism. In conclusion, we confirmed that serum ceramides are positively correlated with markers of both insulin resistance and impaired \\u03b2-cell function. Inflammation is an important driver of ceramide synthesis, and our data suggest that gut dysbiosis, characterized by decreased metagenomic richness, lower abundance of anti-inflammatory bifidobacteria species, and a greater metagenomic capacity for toll-like receptor ligand synthesis, may contribute to this pathway in human obesity. This study paves the way for future experiments aimed at elucidating the role of the gut microbiota in host lipid metabolism and the pathogenesis of type 2 diabetes.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Below is an example of the function using a PubMed search query. Since we are having difficulties running edirect from Colab this is just an example and can not be run here."
      ],
      "metadata": {
        "id": "aJd7lRC9lfMn"
      },
      "id": "aJd7lRC9lfMn"
    },
    {
      "cell_type": "code",
      "source": [
        "\"\"\"\n",
        "bioscraping(\n",
        "    'mecp2 AND human[mesh] AND English[lang] AND (\"2020/04/01\"[Date - Publication] : \"2020/06/30\"[Date - Publication])''',\n",
        "    EMAIL, #You need to insert your email address here\n",
        "    NCBI_API_KEY, #You need to insert your NCBI_API_KEY here\n",
        "    wiley_api_key = WILEY_API_KEY, #This is an optional parameter.\n",
        "    #You can insert your WILEY_API_KEY here\n",
        "    elsevier_api_key = ELSEVIER_API_KEY, #This is an optional parameter.\n",
        "    #You can insert your ELSEVIER_API_KEY here\n",
        "    colab1 = True # This parameter is for this Notebook example only.\n",
        "    #It bypass the edirect module that can not be run on Colab.\n",
        "    )\n",
        "\"\"\""
      ],
      "metadata": {
        "collapsed": true,
        "id": "6kPhfW3olcBn"
      },
      "id": "6kPhfW3olcBn",
      "execution_count": null,
      "outputs": []
    }
  ]
}